---
layout: post
title: Proceedings of RECOMB-Seq
permalink: /proceedings/
isStaticPost: true
---

#### Full Proceedings Papers

##### 2014

  - **Assembly-free genome comparison based on next-generation sequencing reads and variable length patterns.** Matteo Comin and Michele Schimd. [BMC Bioinformatics 15 (Suppl 9) S1 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S1)
  - **SWAP-Assembler: scalable and efficient genome assembly towards thousands of cores.** Jintao Meng, Bingqiang Wang, Yanjie Wei, Shengzhong Feng and Pavan Balaji. [BMC Bioinformatics 15 (Suppl 9) S2 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S2)
  - **Piecing the puzzle together: a revisit to transcript reconstruction problem in RNA-seq.** Yan Huang, Yin Hu and Jinze Liu. [BMC Bioinformatics 15 (Suppl 9) S3 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S3)
  - **Near-optimal assembly for shotgun sequencing with noisy reads.** Ka-Kit Lam, Asif Khalak and David Tse. [BMC Bioinformatics 15 (Suppl 9) S4 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S4)
  - **On the complexity of Minimum Path Cover with Subpath Constraints for multi-assembly.** Romeo Rizzi, Alexandru I Tomescu and Veli Mäkinen. [BMC Bioinformatics 15 (Suppl 9) S5 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S5)
  - **BADGE: A novel Bayesian model for accurate abundance quantification and differential analysis of RNA-Seq data.** Jinghua Gu, Xiao Wang, Leena Halakivi-Clarke, Robert Clarke and Jianhua Xuan. [BMC Bioinformatics 15 (Suppl 9) S6 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S6)
  - **Fast lossless compression via cascading Bloom filters.** Roye Rozov, Ron Shamir and Eran Halperin. [BMC Bioinformatics 15 (Suppl 9) S7 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S7)
  - **Gene finding in metatranscriptomic sequences.** Wazim Mohammed Ismail, Yuzhen Ye and Haixu Tang. [BMC Bioinformatics 15 (Suppl 9) S8 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S8)
  - **ILP-based maximum likelihood genome scaffolding.** James Lindsay, Hamed Salooti, Ion Măndoiu and Alex Zelikovsky. [BMC Bioinformatics 15 (Suppl 9) S9 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S9)
  - **CASPER: context-aware scheme for paired-end reads from high-throughput amplicon sequencing.** Sunyoung Kwon, Byunghan Lee and Sungroh Yoon. [BMC Bioinformatics 15 (Suppl 9) S10 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S10)
  - **PuFFIN - a parameter-free method to build nucleosome maps from paired-end reads.** Anton Polishko, Evelien M Bunnik, Karine G Le Roch and Stefano Lonardi. [BMC Bioinformatics 15 (Suppl 9) S11 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S11)
  - **ARYANA: Aligning Reads by Yet Another Approach.** Milad Gholami, Aryan Arbabi, Ali Sharifi-Zarchi, Hamidreza Chitsaz and Mehdi Sadeghi. [BMC Bioinformatics 15 (Suppl 9) S12 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S12)
  - **Ensemble analysis of adaptive compressed genome sequencing strategies.** Zeinab Taghavi. [BMC Bioinformatics 15 (Suppl 9) S13 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S13)
  - **A better sequence-read simulator program for metagenomics.** Stephen Johnson, Brett Trost, Jeffrey R Long, Vanessa Pittet and Anthony Kusalik. [BMC Bioinformatics 15 (Suppl 9) S14 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S14)
  - **ProbeAlign: incorporating high-throughput sequencing-based structure probing information into ncRNA homology search.** Ping Ge, Cuncong Zhong and Shaojie Zhang. [BMC Bioinformatics 15 (Suppl 9) S15 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S15)
  - **Detecting epigenetic motifs in low coverage and metagenomics settings.** Noam D Beckmann, Sashank Karri, Gang Fang and Ali Bashir. [BMC Bioinformatics 15 (Suppl 9) S16 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S16)


##### 2013

  - **Discovering motifs that induce sequencing errors.** Manuel Allhoff, Alexander Schönhuth, Marcel Martin, Ivan G Costa, Sven Rahmann and Tobias Marschall. [BMC Bioinformatics 14 (Suppl 5) S1 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S1)
  - **metaBEETL: high-throughput analysis of heterogeneous microbial populations from shotgun DNA sequences.** Christina Ander, Ole B Schulz-Trieglaff, Jens Stoye and Anthony J Cox. [BMC Bioinformatics 14 (Suppl 5) S2 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S2)
  - **Joint genotype inference with germline and somatic mutations.** Eric Bareke, Virginie Saillour, Jean-François Spinella, Ramon Vidal, Jasmine Healy, Daniel Sinnett and Miklós Csűrös. [BMC Bioinformatics 14 (Suppl 5) S3 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S3)
  - **Discovering and mapping chromatin states using a tree hidden Markov model.** Jacob Biesinger, Yuanfeng Wang and Xiaohui Xie. [BMC Bioinformatics 14 (Suppl 5) S4 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S4)
  - **Prioritization of candidate disease genes by topological similarity between disease and protein diffusion profiles.** Jie Zhu, Yufang Qin, Taigang Liu, Jun Wang and Xiaoqi Zheng. [BMC Bioinformatics 14 (Suppl 5) S5 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S5)
  - **Bellerophon: a hybrid method for detecting interchromo-somal rearrangements at base pair resolution using next-generation sequencing data.** Matthew Hayes and Jing Li. [BMC Bioinformatics 14 (Suppl 5) S6 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S6)
  - **De Bruijn superwalk with multiplicities problem is NP-hard.** Evgeny Kapun and Fedor Tsarev. [BMC Bioinformatics 14 (Suppl 5) S7 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S7)
  - **A random-permutations-based approach to fast read alignment.** Roy Lederman. [BMC Bioinformatics 14 (Suppl 5) S8 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S8)
  - **Assembling contigs in draft genomes using reversals and block-interchanges.** Chi-Long Li, Kun-Tze Chen and Chin Lung Lu. [BMC Bioinformatics 14 (Suppl 5) S9 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S9)
  - **An optimized algorithm for detecting and annotating regional differential methylation.** Sheng Li, Francine E Garrett-Bakelman, Altuna Akalin, Paul Zumbo, Ross Levine, Bik L To, Ian D Lewis, Anna L Brown, Richard J D'Andrea, Ari Melnick and Christopher E Mason. [BMC Bioinformatics 14 (Suppl 5) S10 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S10)
  - **A mixture model for expression deconvolution from RNA-seq in heterogeneous tissues.** Yi Li and Xiaohui Xie. [BMC Bioinformatics 14 (Suppl 5) S11 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S11)
  - **Gene prediction in metagenomic fragments based on the SVM algorithm.** Yongchu Liu, Jiangtao Guo, Gangqing Hu and Huaiqiu Zhu. [BMC Bioinformatics 14 (Suppl 5) S12 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S12)
  - **Evaluating genome architecture of a complex region via generalized bipartite matching.** Christine Lo, Sangwoo Kim, Shay Zakov and Vineet Bafna. [BMC Bioinformatics 14 (Suppl 5) S13 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S13)
  - **CLASS: constrained transcript assembly of RNA-seq reads.** Li Song and Liliana Florea. [BMC Bioinformatics 14 (Suppl 5) S14 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S14)
  - **A novel min-cost flow method for estimating transcript expression with RNA-Seq.** Alexandru I Tomescu, Anna Kuosmanen, Romeo Rizzi and Veli Mäkinen. [BMC Bioinformatics 14 (Suppl 5) S15 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S15)
  - **Gene set enrichment analysis of RNA-Seq data: integrating differential expression and splicing.** Xi Wang and Murray J Cairns. [BMC Bioinformatics 14 (Suppl 5) S16 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S16)
  - **De novo inference of stratification and local admixture in sequencing studies.** Yu Zhang. [BMC Bioinformatics 14 (Suppl 5) S17 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S17)
  - **Optimal assembly for high throughput shotgun sequencing.** Guy Bresler, Ma'ayan Bresler and David Tse. [BMC Bioinformatics 14 (Suppl 5) S18 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S18)


##### 2012

  - **Exploiting sparseness in *de novo* genome assembly.** Chengxi Ye, Zhanshan Sam Ma, Charles H Cannon, Mihai Pop and Douglas W Yu. [BMC Bioinformatics 13 (Suppl 6), S1 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S1).
  - **MGMR: leveraging RNA-Seq population data to optimize expression estimation.** Roye Rozov, Eran Halperin and Ron Shamir. [BMC Bioinformatics 13 (Suppl 6), S2 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S2).
  - **Haplotype reconstruction using perfect phylogeny and sequence data.** Anatoly Efros and Eran Halperin. [BMC Bioinformatics 13 (Suppl 6), S3 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S3).
  - **Biases in read coverage demonstrated by interlaboratory and interplatform comparison of 117 mRNA and genome sequencing experiments.** Ekaterina E Khrameeva and Mikhail S Gelfand. [BMC Bioinformatics 13 (Suppl 6), S4 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S4).
  - **KIS SPLICE: de-novo calling alternative splicing events from RNA-seq data.** Gustavo AT Sacomoto, Janice Kielbassa, Rayan Chikhi, Raluca Uricaru, Pavlos Antoniou, Marie-France Sagot,   Pierre Peterlongo and Vincent Lacroix. [BMC Bioinformatics 13 (Suppl 6), S5 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S5).
  - **An improved approach for accurate and efficient calling of structural variations with low-coverage sequence data.** Jin Zhang, Jiayin Wang and Yufeng Wu. [BMC Bioinformatics 13 (Suppl 6), S6 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S6).
  - **PAIR: polymorphic *Alu* insertion recognition.** Jón Ingi Sveinbjörnsson and Bjarni V Halldórsson. [BMC Bioinformatics 13 (Suppl 6), S7 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S7).
  - **High-resolution genetic mapping with pooled sequencing.** Matthew D Edwards and David K Gifford. [BMC Bioinformatics 13 (Suppl 6), S8 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S8).
  - **A context-based approach to identify the most likely mapping for RNA-seq experiments.** Thomas Bonfert, Gergely Csaba, Ralf Zimmer and Caroline C Friedel. [BMC Bioinformatics 13 (Suppl 6), S9 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S9).
  - **Reconstructing cancer genomes from paired-end sequencing data.** Layla Oesper, Anna Ritz, Sarah J Aerni, Ryan Drebin and Benjamin J Raphael. [BMC Bioinformatics 13 (Suppl 6), S10 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S10).
  - **Challenges in estimating percent inclusion of alternatively spliced junctions from RNA-seq data.** Boyko Kakaradov, Hui Yuan Xiong, Leo J Lee, Nebojsa Jojic and Brendan J Frey. [BMC Bioinformatics 13 (Suppl 6), S11 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S11).

##### 2011

  - **Inferring viral spectrum from 454 pyrosequencing reads.** Irina Astrovskaya, Bassam Tork, Serghei Mangul, Kelly Westbrooks, Ion Mandoiu, Peter Balfe and Alex Zelikovsky. [BMC Bioinformatics 12 (Suppl 6), S1 (2011)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-S6-S1).
  - **Genotyping common and rare variation using overlapping pool sequencing.** Dan He, Noah Zaitlen, Bogdan Pasaniuc, Eleazar Eskin and Eran Halperin. [BMC Bioinformatics 12 (Suppl 6), S2 (2011)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-S6-S2).
  - **Assembly of non-unique insertion content using next-generation sequencing.** Nathaniel Parrish, Farhad Hormozdiari and Eleazar Eskin. [BMC Bioinformatics 12 (Suppl 6), S3 (2011)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-S6-S3).
  - **A Hidden Markov Model for copy number variant prediction from whole genome resequencing data.** Yufeng Shen, Yiwei Gu and Itsik Pe'Er. [BMC Bioinformatics 12 (Suppl 6), S4 (2011)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-S6-S4).


#### Short Talks Selected from Abstracts

##### 2011

  - **A whole-genome probe design for massively parallel variant validation using selective circularization.** Daniel Newburger, Georges Natsoulis, Hua Xu, Sue Grimes, John Bell and Hanlee Ji.
  - **Accurate estimation of gene expression levels from DGE sequencing data.** Marius Nicolae and Ion Mandoiu.
  - **Constrained traversal of repeats with paired sequences.** Sébastien Boisvert, Élénie Godzaridis, François Laviolette and Jacques Corbeil.
  - **Contig graph mining for duplication breakpoints.** Jurgen F. Nijkamp, Jean-Marc Daran, Marcel J.T. Reinders and Dick De Ridder.
  - **Counting k-mers with a Bloom Filter.** Pall Melsted and Jonathan Pritchard.
  - **Finding deletions with exact break points from noisy low coverage paired-end short sequence reads.** Jin Zhang and Yufeng Wu.
  - **Improved variant discovery and allele frequency estimation from pooled dna resequencing with Bayesian latent class analysis and compositional bias models.** Shom Paul and Aaron Mackey.
  - **Modeling and automation of sequencing-based determination of RNA structure.** Sharon Aviran, Cole Trapnell, Julius Lucks, Stefanie Mortimer, Shujun Luo, Gary Schroth, Jennifer Doudna, Adam Arkin and Lior Pachter.
  - **mTiM: margin-based transcript mapping from RNA-seq.** Georg Zeller, Nico Goernitz, Gunnar Raetsch, Jonas Behr, Andre Kahles, Soeren Sonnenburg and Pramod Mudrakarta.
  - **Separating metagenomic data into genomes via clustering.** Olga Tanaseichuk and Tao Jiang.
  - **TavernaPBS: custom next-generation sequence analysis workflows using high-performance computing resources with Taverna and PBS.** Mark Lawson, Paul Shuber and Aaron Mackey.


#### Posters

##### 2011

  - **Epigenetics of atherosclerosis.** Lauren Mills, Brian Wamhoff, Brett Blackman, Aaron Mackey and Jessica Connelly.
  - **Genome-scale analysis of promoter melting in eukaryotic gene transcription.** Fedor Kouzine, Damian Wójtowicz, Arito Yamane, Wolfgang Resch, Teresa M. Przytycka, David Levens and Rafael Casellas.
  - **Haplotype discovery based on unassembled sequences estimation.** Serghei Mangul and Alex Zelikovsky.
  - **RGASP evaluation of RNA-seq read alignment algorithms.** Andre Kahles, Regina Bohnert, Paolo Ribeca, Jonas Behr and Gunnar Raetsch.
  - **Screening for transposable element-induced adaptations in Drosophila melanogaster using next-gen sequencing data.** Anna-Sophie Fiston-Lavier, Dmitri Petrov and Josefa Gonzalez.
  - **SeqMDD: symbolic data structures for accurate mapping.** Marco Beccuti, Francesca Cordero, Susanna Donatelli and Raffaele Calogero.
  - **SlideSort: A fast and exact tool for finding all similar pairs from next-generation sequencing data.** Kana Shimizu and Koji Tsuda.
  - **Smarti - A fast short read alignment algorithm.** Florian Schatz, Sascha Möller and Manfred Schimmler.
  - **Toward assessing the quality of *de novo* assembly.** Rasiah Loganantharaj.

