---
layout: post
title: Proceedings of RECOMB-Seq
permalink: /proceedings/
isStaticPost: true
---

#### Full Proceedings Papers

##### 2012

  - **Exploiting sparseness in *de novo* genome assembly. Chengxi Ye, Zhanshan Sam Ma, Charles H Cannon, Mihai Pop and Douglas W Yu.** [BMC Bioinformatics 13 (Suppl 6), S1 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S1).
  - **MGMR: leveraging RNA-Seq population data to optimize expression estimation.** Roye Rozov, Eran Halperin and Ron Shamir. [BMC Bioinformatics 13 (Suppl 6), S2 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S2).
  - **Haplotype reconstruction using perfect phylogeny and sequence data.** Anatoly Efros and Eran Halperin. [BMC Bioinformatics 13 (Suppl 6), S3 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S3).
  - **Biases in read coverage demonstrated by interlaboratory and interplatform comparison of 117 mRNA and genome sequencing experiments.** Ekaterina E Khrameeva and Mikhail S Gelfand. [BMC Bioinformatics 13 (Suppl 6), S4 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S4).
  - **KIS SPLICE: de-novo calling alternative splicing events from RNA-seq data.** Gustavo AT Sacomoto, Janice Kielbassa, Rayan Chikhi, Raluca Uricaru, Pavlos Antoniou, Marie-France Sagot,   Pierre Peterlongo and Vincent Lacroix. [BMC Bioinformatics 13 (Suppl 6), S5 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S5).
  - **An improved approach for accurate and efficient calling of structural variations with low-coverage sequence data.** Jin Zhang, Jiayin Wang and Yufeng Wu. [BMC Bioinformatics 13 (Suppl 6), S6 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S6).
  - **PAIR: polymorphic *Alu* insertion recognition.** Jón Ingi Sveinbjörnsson and Bjarni V Halldórsson. [BMC Bioinformatics 13 (Suppl 6), S7 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S7).
  - **High-resolution genetic mapping with pooled sequencing.** Matthew D Edwards and David K Gifford. [BMC Bioinformatics 13 (Suppl 6), S8 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S8).
  - **A context-based approach to identify the most likely mapping for RNA-seq experiments.** Thomas Bonfert, Gergely Csaba, Ralf Zimmer and Caroline C Friedel. [BMC Bioinformatics 13 (Suppl 6), S9 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S9).
  - **Reconstructing cancer genomes from paired-end sequencing data.** Layla Oesper, Anna Ritz, Sarah J Aerni, Ryan Drebin and Benjamin J Raphael. [BMC Bioinformatics 13 (Suppl 6), S10 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S10).
  - **Challenges in estimating percent inclusion of alternatively spliced junctions from RNA-seq data.** Boyko Kakaradov, Hui Yuan Xiong, Leo J Lee, Nebojsa Jojic and Brendan J Frey. [BMC Bioinformatics 13 (Suppl 6), S11 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S11).

##### 2011

  - **Inferring viral spectrum from 454 pyrosequencing reads.** Irina Astrovskaya, Bassam Tork, Serghei Mangul, Kelly Westbrooks, Ion Mandoiu, Peter Balfe and Alex Zelikovsky. [BMC Bioinformatics 12 (Suppl 6), S1 (2011)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-S6-S1).
  - **Genotyping common and rare variation using overlapping pool sequencing.** Dan He, Noah Zaitlen, Bogdan Pasaniuc, Eleazar Eskin and Eran Halperin. [BMC Bioinformatics 12 (Suppl 6), S2 (2011)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-S6-S2).
  - **Assembly of non-unique insertion content using next-generation sequencing.** Nathaniel Parrish, Farhad Hormozdiari and Eleazar Eskin. [BMC Bioinformatics 12 (Suppl 6), S3 (2011)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-S6-S3).
  - **A Hidden Markov Model for copy number variant prediction from whole genome resequencing data.** Yufeng Shen, Yiwei Gu and Itsik Pe'Er. [BMC Bioinformatics 12 (Suppl 6), S4 (2011)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-S6-S4).


#### Short Talks Selected from Abstracts

##### 2011

  - **A whole-genome probe design for massively parallel variant validation using selective circularization.** Daniel Newburger, Georges Natsoulis, Hua Xu, Sue Grimes, John Bell and Hanlee Ji.
  - **Accurate estimation of gene expression levels from DGE sequencing data.** Marius Nicolae and Ion Mandoiu.
  - **Constrained traversal of repeats with paired sequences.** Sébastien Boisvert, Élénie Godzaridis, François Laviolette and Jacques Corbeil.
  - **Contig graph mining for duplication breakpoints.** Jurgen F. Nijkamp, Jean-Marc Daran, Marcel J.T. Reinders and Dick De Ridder.
  - **Counting k-mers with a Bloom Filter.** Pall Melsted and Jonathan Pritchard.
  - **Finding deletions with exact break points from noisy low coverage paired-end short sequence reads.** Jin Zhang and Yufeng Wu.
  - **Improved variant discovery and allele frequency estimation from pooled dna resequencing with Bayesian latent class analysis and compositional bias models.** Shom Paul and Aaron Mackey.
  - **Modeling and automation of sequencing-based determination of RNA structure.** Sharon Aviran, Cole Trapnell, Julius Lucks, Stefanie Mortimer, Shujun Luo, Gary Schroth, Jennifer Doudna, Adam Arkin and Lior Pachter.
  - **mTiM: margin-based transcript mapping from RNA-seq.** Georg Zeller, Nico Goernitz, Gunnar Raetsch, Jonas Behr, Andre Kahles, Soeren Sonnenburg and Pramod Mudrakarta.
  - **Separating metagenomic data into genomes via clustering.** Olga Tanaseichuk and Tao Jiang.
  - **TavernaPBS: custom next-generation sequence analysis workflows using high-performance computing resources with Taverna and PBS.** Mark Lawson, Paul Shuber and Aaron Mackey.


#### Posters

##### 2011

  - **Epigenetics of atherosclerosis.** Lauren Mills, Brian Wamhoff, Brett Blackman, Aaron Mackey and Jessica Connelly.
  - **Genome-scale analysis of promoter melting in eukaryotic gene transcription.** Fedor Kouzine, Damian Wójtowicz, Arito Yamane, Wolfgang Resch, Teresa M. Przytycka, David Levens and Rafael Casellas.
  - **Haplotype discovery based on unassembled sequences estimation.** Serghei Mangul and Alex Zelikovsky.
  - **RGASP evaluation of RNA-seq read alignment algorithms.** Andre Kahles, Regina Bohnert, Paolo Ribeca, Jonas Behr and Gunnar Raetsch.
  - **Screening for transposable element-induced adaptations in Drosophila melanogaster using next-gen sequencing data.** Anna-Sophie Fiston-Lavier, Dmitri Petrov and Josefa Gonzalez.
  - **SeqMDD: symbolic data structures for accurate mapping.** Marco Beccuti, Francesca Cordero, Susanna Donatelli and Raffaele Calogero.
  - **SlideSort: A fast and exact tool for finding all similar pairs from next-generation sequencing data.** Kana Shimizu and Koji Tsuda.
  - **Smarti - A fast short read alignment algorithm.** Florian Schatz, Sascha Möller and Manfred Schimmler.
  - **Toward assessing the quality of *de novo* assembly.** Rasiah Loganantharaj.

