---
layout: post
title: Proceedings of RECOMB-Seq
permalink: /proceedings/
isStaticPost: true
---

[RECOMB-Seq Google Scholar Profile](https://scholar.google.com/citations?hl=en&user=GrrEQ5QAAAAJ){:target="_blank"}

#### Full Proceedings Papers

##### 2025

  - **Alignment-free detection of differences between sequencing data sets.** Alessia Petescia, Luca Denti, Askar Gafurov, Viktória Hodorová, Jozef Nosek, Broňa Brejová and Tomas Vinar. [iScience, 113828, (2025)](https://doi.org/10.1016/j.isci.2025.113828) 
  - **Tile-X: A vertex reordering approach for scalable long read assembly.** Oieswarya Bhowmik and Ananth Kalyanaraman. [bioRxiv 2025.04.21.649853, (2025)](https://www.biorxiv.org/content/10.1101/2025.04.21.649853v1.full)
  - **Decoding the causal drivers of spatial cellular topology.** Prannav Shankar, Huan Liang, Uthsav Chitra and Rohit Singh. [bioRxiv 2025.03.19.644241, (2025)](https://www.biorxiv.org/content/10.1101/2025.03.19.644241v2)
  - **Efficient trace reconstruction in DNA storage systems using Bidirectional Beam Search.** Zhenhao Gu, Hongyi Xin, Puru Sharma, Gary Yipeng Goh, Limsoon Wong and Niranjan Nagarajan. [bioRxiv 2025.04.16.644694, (2025)](https://www.biorxiv.org/content/10.1101/2025.04.16.644694v1)
  - **Decoupling Hi-C matrices with non-negative matrix factorization to reveal assorted TADs.** Zhao Ling, Shi Ying Li, Jingwan Wang, Xikang Feng, Bowen Tan and Shuai Cheng Li. 
  - **Run-length compressed metagenomic read classification with SMEM-finding and tagging.** Lore Depuydt, Omar Y. Ahmed, Jan Fostier, Ben Langmead and Travis Gagie. [bioRxiv 2025.02.25.640119, (2025)](https://www.biorxiv.org/content/10.1101/2025.02.25.640119v2)
  - **De novo clustering of extensive long-read transcriptome datasets with isONclust3.** Alexander Petri and Kristoffer Sahlin. [overlay preprint](https://doi.org/10.1101/2024.10.29.620862)
  - **Multi-context seeds enable fast and high-accuracy read mapping.** Ivan Tolstoganov, Marcel Martin and Kristoffer Sahlin. [overlay preprint](https://doi.org/10.1101/2024.10.29.620855)
  - **GPU-accelerated homology search with MMseqs2.** Felix Kallenborn, Alejandro Chacon, Christian Hundt, Hassan Sirelkhatim, Kieran Didi, Sooyoung Cha, Christian Dallago, Milot Mirdita, Bertil Schmidt and Martin Steinegger. [overlay preprint](https://doi.org/10.1101/2024.11.13.623350)
  - **DeepMM: identify and correct metagenome misassemblies with deep learning.** Yi Ding, Jin Xiao, Bohao Zou and Lu Zhang. [overlay preprint](https://doi.org/10.1101/2025.02.07.637187)
  - **On the coverage required for diploid genome assembly.** Daanish Mahajan, Chirag Jain and Navin Kashyap. [overlay preprint](https://doi.org/10.48550/arXiv.2405.05734)
  - **Pangenome graph augmentation from unassembled long reads.** Luca Denti, Paola Bonizzoni, Brona Brejova, Rayan Chikhi, Thomas Krannich, Tomas Vinar and Fereydoun Hormozdiari. [overlay preprint](https://doi.org/10.1101/2025.02.07.637057)

##### 2024

  - **Movi: a fast and cache-efficient full-text pangenome index.** Mohsen Zakeri, Nathaniel Brown, Omar Ahmed, Travis Gagie and Ben Langmead. [iScience, 27 (12): 111464, (2024)](https://www.sciencedirect.com/science/article/pii/S2589004224026919)
  - **Building a Pangenome Alignment Index via Recursive Prefix-Free Parsing.** Eddie Ferro, Marco Oliva, Travis Gagie and Christina Boucher. [iScience, 27 (10): 110933, (2024)](https://www.sciencedirect.com/science/article/pii/S2589004224021588)
  - **The Backpack Quotient Filter: a dynamic and space-efficient data structure for querying k-mers with abundance.** Victor Levallois, Pierre Peterlongo, Yoann Dufresne, Bertrand Le Gal and Francesco Andreace. [iScience, 27 (12): 111435, (2024)](https://www.sciencedirect.com/science/article/pii/S2589004224026609)
  - **Synthetic promoter design in Escherichia coli based on multinomial diffusion model.** Qixiu Du, May Nee Poon, Xiaocheng Zeng, Pengcheng Zhang, Zheng Wei, Haochen Wang, Ye Wang, Lei Wei and Xiaowo Wang. [iScience, 27 (11): 111207, (2024)](https://www.sciencedirect.com/science/article/pii/S2589004224024325)
  - **Quick and effective approximation of in silico saturation mutagenesis experiments with first-order Taylor expansion.** Alexander Sasse, Sara Mostafavi and Maria Chikina. [iScience, 27 (9): 110807, (2024)](https://www.sciencedirect.com/science/article/pii/S2589004224020327)
  - **Flowtigs: safety in flow decompositions for assembly graphs.** Francisco Sena, Eliel Ingervo, Shahbaz Khan, Andrey Prjibelski, Sebastian Schmidt and Alexandru Tomescu. [iScience, 27 (12): 111208, (2024)](https://www.sciencedirect.com/science/article/pii/S2589004224024337)
 - **PLA-complexity of k-mer multisets.** Md Hasin Abrar and Paul Medvedev. [overlay preprint](https://doi.org/10.1101/2024.02.08.579510)
  - **Phasing Diploid Genome Assembly Graphs with Single-Cell Strand Sequencing.** Mir Henglin, Maryam Ghareghani, William Harvey, David Porubsky, Sergey Koren, Evan Eichler, Peter Ebert and Tobias Marschall. [overlay preprint](https://doi.org/10.1101/2024.02.15.580432)
  - **CNVeil enables accurate and robust tumor subclone identification and copy number estimation from single cell sequencing data.** Weiman Yuan, Can Luo, Yunfei Hu, Liting Zhang, Zihang Wen, Yichen Henry Liu, Xian Mallory and Xin Maizie Zhou. [overlay preprint](https://doi.org/10.1101/2024.02.21.581409)
  - **Quality assessment of splice site annotation based on conservation across multiple species.** Ilia Minkin and Steven Salzberg. [overlay preprint](https://doi.org/10.1101/2023.12.01.569581)
  - **MaskGraphene: Advancing joint embedding, clustering, and batch correction for spatial transcriptomics using graph-based self-supervised learning.** Yunfei Hu, Yikang Li, Manfei Xie, Mingxing Rao, Yichen Henry Liu and Xin Maizie Zhou. [overlay preprint](https://doi.org/10.1101/2024.02.21.581387)
  - **Investigating the performance of foundation models on human 3’UTR sequences.** Sergey Vilov and Matthias Heinig. [overlay preprint](https://doi.org/10.1101/2024.02.09.579631)

##### 2023

  - **TreeTerminus - creating transcript trees using inferential replicate counts.** Noor Pratap Singh, Michael I Love and Rob Patro. [iScience, 26 (6): 106961, (2023)](https://www.sciencedirect.com/science/article/pii/S2589004223010386)
  - **DandD: efficient measurement of sequence growth and similarity.** Jessica Bonnie, Omar Ahmed and Ben Langmead. [iScience, 27 (3): 109054, (2024)](https://www.sciencedirect.com/science/article/pii/S258900422400275X)
  - **ADEPT: autoencoder with differentially expressed genes and imputation for a robust spatial transcriptomics clusterin.** Yunfei Hu, Yuying Zhao, Curtis Schunk, Yingxiang Ma, Tyler Derr and Xin Maizie Zhou. [iScience, 26 (6): 106792, (2023)](https://www.sciencedirect.com/science/article/pii/S2589004223008696)
  - **aKmerBroom: ancient oral DNA decontamination using Bloom filters on k-mer sets.** Camila Duitama González, Riccardo Vicedomini, Hugues Richard and Rayan Chikhi. [iScience, 26 (11): 108057, (2023)](https://www.sciencedirect.com/science/article/pii/S258900422302134X)
  - **Masked superstrings as a unified framework for textual k-mer set representations.** Ondrej Sladky, Pavel Vesely and Karel Brinda. [bioRxiv 2023.02.01.526717, (2023)](https://www.biorxiv.org/content/10.1101/2023.02.01.526717v1)
  - **WGT: tools and algorithms for recognizing, visualizing and generating Wheeler graphs.** Kuan-Hao Chao, Pei-Wei Chen, Sanjit A. Seshia and Ben Langmead. [iScience, 26 (8): 107402, (2023)](https://www.sciencedirect.com/science/article/pii/S2589004223014797)
  - **Exploring high-quality microbial genomes by assembly of linked-reads with high barcode specificity using deep learning.** Zhenmiao Zhang, Hongbo Wang, Chao Yang, Yufen Huang, Zhen Yue, Yang Chen, Lijuan Han, Aiping Lyu, Xiaodong Fang and Lu Zhang. [overlay preprint](https://doi.org/10.1101/2022.09.07.506963)
  - **A safety framework for flow decomposition problems via integer linear programming.** Fernando Hugo Cunha Dias, Manuel Caceres, Lucia Williams, Brendan Mumey and Alexandru I. Tomescu. [overlay preprint](https://doi.org/10.48550/arXiv.2301.13245)
  - **Coverage-preserving sparsification of overlap graphs for long-read assembly.** Chirag Jain. [overlay preprint](https://doi.org/10.1101/2022.03.17.484715 )
  - **Predicting transcription factor activity using prior biological information.** Joseph Estabrook, William Yashar, Hannah Holly, Julia Somers, Olga Nikolova, Ozgun Babur, Theodore Braun and Emek Demir. [overlay preprint](https://doi.org/10.1101/2022.12.16.520295)
  - **On minimizers and convolutional filters: a partial justification for the effectiveness of CNNs in categorical sequence analysis.** Yun William Yu. [overlay preprint](https://doi.org/10.48550/arXiv.2111.08452)
  - **TargetCall: eliminating the wasted computation in basecalling via pre-basecalling filtering.** Meryem Banu Cavlak, Gagandeep Singh, Mohammed Alser, Can Firtina, Joël Lindegger, Mohammad Sadrosadati, Nika Mansouri Ghiasi, Can Alkan and Onur Mutlu. [overlay preprint](https://doi.org/10.48550/arXiv.2212.04953)
  - **An intrinsically interpretable neural network architecture for sequence to function learning.** Ali Tugrul Balci, Mark Maher Ebeid, Panayiotis Benos, Dennis Kostka and Maria Chikina. [overlay preprint](https://doi.org/10.1101/2023.01.25.525572)
  - **Scrooge: a fast and memory-frugal genomic sequence aligner for CPUs, GPUs, and ASICs.** Joël Lindegger, Damla Senol Cali, Mohammed Alser, Juan Gómez-Luna, Nika Mansouri Ghiasi and Onur Mutlu. [overlay preprint](https://doi.org/10.48550/arXiv.2208.09985)


##### 2022

  - **Sequence-based pangenomic core detection.** Tizian Schulz, Roland Wittler and Jens Stoye. [iScience, 25 (6): 104413, (2022)](https://www.sciencedirect.com/science/article/pii/S2589004222006848)
  - **scTagger: Fast and accurate matching of cellular barcodes across short- and long-reads of single-cell RNA-seq experiments.** Ghazal Ebrahimi, Baraa Orabi, Meghan Robinson, Cedric Chauve, Ryan Flannigan and Faraz Hach. [iScience, 25 (7): 104530, (2022)](https://www.sciencedirect.com/science/article/pii/S258900422200801X)
  - **Genetic polyploid phasing from low-depth progeny samples.** Sven Schrinner, Rebecca Serra Mari, Richard Finkers, Paul Arens, Björn Usadel, Tobias Marschall and Gunnar W. Klau. [iScience, 25 (6): 104461, (2022)](https://www.sciencedirect.com/science/article/pii/S2589004222007325)
  - **BOA: a partitioned view of genome assembly .** Priyanka Ghosh, Xiaojing An, Patrick Keppler, Sureyya Emre Kurt, Umit Catalyurek, Sriram Krishnamoorthy, Ponnuswamy Sadayappan, Aravind Sukumaran Rajam and Ananth Kalyanaraman. [iScience, 25 (11): 105273, (2022)](https://www.sciencedirect.com/science/article/pii/S2589004222015450)
  - **Mapping-friendly sequence reductions: going beyond homopolymer compression.** Luc Blassel, Paul Medvedev and Rayan Chikhi. [iScience, 25 (11): 105305, (2022)](https://www.sciencedirect.com/science/article/pii/S2589004222015772)
  - **BinSPreader: refine binning results for fuller MAG reconstruction .** Ivan Tolstoganov, Yury Kamenev, Roman Kruglikov, Sofia Ochkalova and Anton Korobeynikov. [iScience, 25 (8): 104770, (2022)](https://www.sciencedirect.com/science/article/pii/S2589004222010422)
  - **Flexible seed size enables ultra-fast and accurate read alignment.** Kristoffer Sahlin. [overlay preprint](https://doi.org/10.1101/2021.06.18.449070)
  - **Sparse and skew hashing of k-mers.** Giulio Ermanno Pibiri. [overlay preprint](https://doi.org/10.1101/2022.01.15.476199)
  - **Metagenome assembly of high-fidelity long reads with hifiasm-meta.** Xiaowen Feng, Haoyu Cheng, Daniel Portik and Heng Li. [overlay preprint]()
  - **Scalable sequence database search using Partitioned Aggregated Bloom Comb-Trees.** Camille Marchet and Antoine Limasset. [overlay preprint](https://doi.org/10.1101/2022.02.11.480089)
  - **Improved structural variant discovery in hard-to-call regions using sample-specific string detection from accurate long reads.** Luca Denti, Parsoa Khorsand, Paola Bonizzoni, Fereydoun Hormozdiari and Rayan Chikhi. [overlay preprint](https://doi.org/10.1101/2022.02.12.480198)
  - **The minimizer Jaccard estimator is biased and inconsistent.** Mahdi Belbasi, Antonio Blanca, Robert S. Harris, David Koslicki and Paul Medvedev. [overlay preprint](https://doi.org/10.1101/2022.01.14.476226)


##### 2021

  - **Dynamic partitioning of search patterns for approximate pattern matching using search schemes.** Luca Renders, Kathleen Marchal and Jan Fostier. [iScience, 24 (7): 102687, (2021)](https://www.cell.com/iscience/fulltext/S2589-0042(21)00655-6)
  - **Linearization of genome sequence graphs revisited.** Anna Lisiecka and Norbert Dojer. [iScience, 24 (7): 102755, (2021)](https://www.cell.com/iscience/fulltext/S2589-0042(21)00723-9)
  - **Pan-genomic matching statistics for targeted nanopore sequencing.** Omar Ahmed, Massimiliano Rossi, Sam Kovaka, Michael Schatz, Travis Gagie, Christina Boucher and Ben Langmead. [iScience, 24 (6): 102696, (2021)](https://www.cell.com/iscience/fulltext/S2589-0042(21)00664-7)
  - **A molecular taxonomy of tumors independent of tissue-of-origin.** Peter Nguyen, Simon Coetzee, Daniel Lakeland and Dennis Hazelett . [iScience, 24 (10): 103084, (2021)](https://www.cell.com/iscience/fulltext/S2589-0042(21)01052-X)
  - **A reference-free approach for cell type classification with scRNA-seq.** Qi Sun, Yifan Peng and Jinze Liu. [iScience, 24 (8): 102855, (2021)](https://www.cell.com/iscience/fulltext/S2589-0042(21)00823-3)
  - **Accurate spliced alignment of long RNA sequencing reads.** Kristoffer Sahlin and Veli Mäkinen. [overlay preprint](https://doi.org/10.1101/2020.09.02.279208)
  - **Algorithms and complexity on indexing founder graphs.** Massimo Equi, Tuukka Norri, Jarno Alanko, Bastien Cazaux, Alexandru I. Tomescu and Veli Mäkinen. [overlay preprint](https://doi.org/10.48550/arXiv.2102.12822)
  - **HolistIC: leveraging Hi-C and whole genome shotgun sequencing for double minute chromosome discovery.** Matthew Hayes, Angela Nguyen, Rahib Islam, Caryn Butler, Ethan Tran, Derrick Mullins and Chindo Hicks. [overlay preprint](https://doi.org/10.1101/2021.03.22.435038)
  - **CONSULT: Accurate contamination removal usinglocality-sensitive hashing.** Eleonora Rachtman, Vineet Bafna and Siavash Mirarab. [overlay preprint](https://doi.org/10.1101/2021.03.18.436035)
  - **DeepDRIM: a deep neural network to reconstruct cell-type-specific gene regulatory network using single-cell RNA-seq data.** Jiaxing Chen, Chinwang Cheong, Liang Lan, Xin Zhou, Jiming Liu, Aiping Lyu, William K Cheung and Lu Zhang. [overlay preprint](https://doi.org/10.1101/2021.02.03.429484)
  - **Conditional resampling improves calibration and sensitivity in single-cell CRISPR screen analysis.** Timothy Barry, Xuran Wang, Kathryn Roeder and Eugene Katsevich. [overlay preprint](https://doi.org/10.1101/2020.08.13.250092)
  - **Accel-Align: a fast sequence mapper and aligner based on the seed–embed–extend method.** Yiqing Yan, Nimisha Chaturvedi and Raja Appuswamy. [overlay preprint](https://doi.org/10.1101/2020.07.20.211888)
  - **BiTSC2: Bayesian inference of tumor clonal tree by joint analysis of single-cell SNV and CNA data.** Ziwei Chen, Fuzhou Gong, Lin Wan and Liang Ma. [overlay preprint](https://doi.org/10.1101/2020.11.30.380949)
  - **Automated strain separation in low-complexity metagenomes using long reads.** Riccardo Vicedomini, Christopher Quince, Aaron Darling and Rayan Chikhi. [overlay preprint](https://doi.org/10.1101/2021.02.24.429166)
  - **Real-time, direct classification of nanopore signals with SquiggleNet.** Yuwei Bao, Jack Wadden, John Erb-Downward, Piyush Ranjan, Robert Dickson, David Blaauw and Joshua Welch. [overlay preprint](https://doi.org/10.1101/2021.01.15.426907)
  - **Population-scale detection of non-reference sequence variants using colored de Bruijn graphs.** Thomas Krannich, W. Timothy J. White, Sebastian Niehus, Guillaume Holley, Bjarni V. Halldorsson and Birte Kehr. [overlay preprint](https://doi.org/10.1101/2021.03.23.436560)
  - **Ariadne: barcoded linked-read deconvolution using de Bruijn graphs.** Lauren Mak, Dmitry Meleshko, David C. Danko, Waris Barakzai, Natan Belchikov and Iman Hajirasouliha. [overlay preprint](https://doi.org/10.1101/2021.05.09.443255)
  - **ACTIVA: realistic single-cell RNA-seq generation with automatic cell-type identification using introspective variational autoencoders.** A. Ali Heydari, Oscar Davalos, Lihong Zhao, Katrina Hoyer and Suzanne Sindi. [overlay preprint](https://doi.org/10.1093/bioinformatics/btac095)

##### 2020

  - **Spherical thresholding improves sketching of single-cell transcriptomic heterogeneity.** Van Hoan Do, Khaled Elbassioni and Stefan Canzar. [iScience, 23(6): 101126, 2020 S1](https://www.cell.com/iscience/fulltext/S2589-0042(20)30311-4)
  - **HASLR: Fast Hybrid Assembly of Long Reads.** Ehsan Haghshenas, Hossein Asghari, Jens Stoye, Cedric Chauve and Faraz Hach. [iScience, 23(8): 101389, 2020 S2](https://www.cell.com/iscience/fulltext/S2589-0042(20)30577-0)
  - **BATMAN: fast and accurate integration of single-cell RNA-Seq datasets via minimum-weight matching.** Igor Mandric, Brian Hill, Malika Freund, Michael Thompson and Eran Halperin. [iScience 23(6): 101185, 2020 S3](https://www.cell.com/iscience/fulltext/S2589-0042(20)30370-9)
  - **Scalable pairwise whole-genome homology mapping of large genomes with BubbZ.** Ilia Minkin and Paul Medvedev. [iScience 23(6): 101224, 2020 S4](https://www.cell.com/iscience/fulltext/S2589-0042(20)30409-0)
  - **Hierarchically labeled database indexing allows scalable characterization of microbiomes.** Filippo Utro, Niina Haiminen, Enrico Siragusa, Laura-Jayne Gardiner, Edward Seabolt, Ritesh Krishna, James Kaufman and Laxmi Parida. [iScience 23(4): 100988, 2020 S5](https://www.cell.com/iscience/fulltext/S2589-0042(20)30172-3)
  - **Maximal Perfect Haplotype Blocks with Wildcards.** Lucia Williams and Brendan Mumey. [iScience 23(6): 101149, 2020 S6](https://www.cell.com/iscience/fulltext/S2589-0042(20)30334-5)
  - **CONNET: Accurate Diploid Genome Consensus in de novo Assembly of Nanopore Sequencing Data via Deep Learning.** Yifan Zhang, Chi-Man Liu, Henry Cm Leung, Ruibang Luo and Tak-Wah Lam. [iScience 23(5): 101128, 2020 S7](https://www.cell.com/iscience/fulltext/S2589-0042(20)30313-8)
  - **TALC: Transcription-aware long read correction.** Lucile Broseus, Aubin Thomas and William Ritchie. [overlay preprint](https://doi.org/10.1101/2020.01.10.901728)
  - **Metalign: Efficient alignment-based metagenomic profiling via containment min hash.** Nathan Lapierre, Mohammed Alser, Eleazar Eskin, David Koslicki and Serghei Mangul. [overlay preprint](https://doi.org/10.1101/2020.01.17.910521)
  - **SCAPP: An algorithm for improved plasmid assembly in metagenomes.** David Pellow, Maraike Probst, Ori Furman, Alvah Zorea, Arik Segal, Itzik Mizrahi and Ron Shamir. [overlay preprint](https://doi.org/10.1101/2020.01.12.903252)

##### 2019

  - **GenMap: Fast and Exact Computation of Genome Mappability.** Christopher Pockrandt, Mai Alzamel, Costas Iliopoulos and Knut Reinert. [Bioinformatics, 36(12): 3687-3692, 2020](https://academic.oup.com/bioinformatics/article/36/12/3687/5815974)
  - **Samovar: Single-sample mosaic SNV calling with linked reads.** Charlotte A. Darby, James R. Fitch, Patrick J. Brennan, Benjamin J. Kelly, Natalie Bir, Vincent Magrini, Jeffrey Leonard, Catherine E. Cottrell, Julie M. Gastier-Foster, Richard K. Wilson, Elaine R. Mardis, Peter White, Ben Langmead and Michael C. Schatz . [iScience, 18: 1-10, 2019](https://www.cell.com/iscience/fulltext/S2589-0042(19)30174-9)
  - **PathRacer: racing profile HMM paths on assembly graph.** Alexander Shlemov and Anton Korobeynikov. [bioRxiv 562579](https://www.biorxiv.org/content/10.1101/562579v1)
  - **Accurate and Efficient Mapping of the Cross-linked microRNA-mRNA Duplex Reads.** Cuncong Zhong and Shaojie Zhang. [iScience, 18: 11-19, 2019](https://www.sciencedirect.com/science/article/pii/S2589004219301750?via%3Dihub)
  - **MALVA: genotyping by Mapping-free ALlele detection of known VAriants.** Giulia Bernardini, Paola Bonizzoni, Luca Denti, Marco Previtali and Alexander Schönhuth. [iScience, 18: 20-27, 2019](https://www.sciencedirect.com/science/article/pii/S2589004219302366)
  - **Kevlar: a mapping-free framework for accurate discovery of de novo variants.** Daniel Standage, C. Titus Brown and Fereydoun Hormozdiari. [iScience, 18: 28-36, 2019](https://www.sciencedirect.com/science/article/pii/S2589004219302597)
  - **Parallel decompression of gzip-compressed files and random access to DNA sequences.** Maël Kerbiriou and Rayan Chikhi. [overlay preprint](https://doi.org/10.48550/arXiv.1905.07224)
  - **Indexing De Bruijn graphs with minimizers.** Camille Marchet, Maël Kerbiriou and Antoine Limasset. [overlay preprint](https://doi.org/10.1101/546309)
  - **CONSENT: Scalable self-correction of long reads with multiple sequence alignment.** Pierre Morisse, Camille Marchet, Antoine Limasset, Thierry Lecroq and Arnaud Lefebvre. [overlay preprint](https://doi.org/10.1101/546630)
  - **Graph analysis of fragmented long-read bacterial genome assemblies.** Pierre Marijon, Rayan Chikhi and Jean-Stéphane Varré. [overlay preprint](https://doi.org/10.1093/bioinformatics/btz219)
  - **Improved Representation of Sequence Bloom Trees.** Robert S. Harris and Paul Medvedev. [overlay preprint](https://doi.org/10.1101/501452)
  - **Scalable multiple whole-genome alignment and locally collinear block construction with SibeliaZ.** Ilia Minkin and Paul Medvedev. [overlay preprint](https://doi.org/10.1101/548123)
  - **Nebula: Ultra-efficient mapping free structural variant genotyper.** Parsoa Khorsand and Fereydoun Hormozdiari. [overlay preprint](https://doi.org/10.1101/566620)


##### 2018

  - **Full-length de novo viral quasispecies assembly through variation graph construction.** Jasmijn Baaijens, Bastiaan van der Roest, Johannes Koster, Leen Stougie and Alexander Schoenhuth. [Bioinformatics, 35 (24): 5086-5094, (2019)](https://academic.oup.com/bioinformatics/article/35/24/5086/5506652)
  - **Alignment-free clustering of barcode (UMI) tagged DNA molecules.** Baraa Orabi, Emre Erhan, Brian McConeghy, Stanislav V. Volik, Stephane Le Bihan, Collin C. Collins, Cedric Chauve and Faraz Hach. [Bioinformatics, 35 (11): 1829-1836, (2019)](https://academic.oup.com/bioinformatics/article/35/11/1829/5142725)
  - **Toward perfect reads: self-correction of short reads via mapping on de Bruijn graphs.** Antoine Limasset, Jean-François Flot and Pierre Peterlongo. [Bioinformatics, 36 (5): 1374-1381, (2020)](https://academic.oup.com/bioinformatics/article/36/5/1374/5345559)
  - **TNER: a novel background error suppression method for mutation detection in circulating tumor DNA.** Shibing Deng, Maruja Lira, Donghui Huang, Kai Wang, Crystal Valdez, Jennifer Kinong, Paul Rejto, Jadwiga Bienkowska, James Hardwick and Tao Xie. [BMC Bioinformatics, 18: 387, (2018)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2428-3)
  - **Optimum search schemes for approximate string matching using bidirectional FM-Index.** Kiavash Kianfar, Christopher Pockrandt, Bahman Torkamandi, Haochen Luo and Knut Reinert. [bioRxiv 301085, (2018)](https://www.biorxiv.org/content/10.1101/301085v2)
  - **Resistome SNP calling via read colored de Bruijn graphs .** Bahar Alipanahi, Martin Muggli, Musa Jundi, Noelle Noyes and Christina Boucher. [Bioinformatics, 36 (22-23): 5275-5281, (2021)](https://academic.oup.com/bioinformatics/article/36/22-23/5275/5734643)
  - **Deconvoluting the diversity of within-host pathogen strains in a Multi-Locus Sequence Typing framework.** Guo Liang Gan, Elijah Willie, Cedric Chauve and Leonid Chindelevitch. [BMC Bioinformatics, 20: 637, (2019)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3204-8)

##### 2017

  - **QUENTIN: accurate reconstruction of disease transmissions from viral quasispecies genomic data.** Pavel Skums, Alex Zelikovsky, Sergey Knyazev, Igor Mandric, Zoya Dimitrova, Sumathi Ramachandran, David Stiven Campo Rendon, Leonid Bunimovich, Elizabeth Costenbader, Connie Sexton, Siobhan O'Connor, Guo-Liang Xia and Yury Khudyakov. [Bioinformatics, 34 (1): 163-170, (2018)](https://academic.oup.com/bioinformatics/article/34/1/163/3870483)
  - **An accurate algorithm for the detection of DNA fragments from dilution pool sequencing experiments.** Vikas Bansal. [Bioinformatics, 34 (1): 155-162, (2018)](https://academic.oup.com/bioinformatics/article/34/1/155/3952670)
  - **A novel data structure to support ultra-fast taxonomic classification of metagenomic sequences with k-mer signatures.** Xinan Liu, Ye Yu, Jinpeng Liu, Chen Qian and Jinze Liu. [Bioinformatics, 34 (1): 171-178, (2018)](https://academic.oup.com/bioinformatics/article/34/1/171/3931858)
  - **DIMM-SC: A Dirichlet mixture model for clustering droplet-based single cell transcriptomic data.** Zhe Sun, Ting Wang, Ke Deng, Xiao-Feng Wang, Robert Lafyatis, Ying Ding, Ming Hu and Wei Chen. [Bioinformatics, 34 (1): 139-146, (2018)](https://academic.oup.com/bioinformatics/article/34/1/139/4060554)
  - **Transcription factor-8mer glossary with GENRE genomic background enables precise identification of lineage-specific co-regulators.** Luca Mariani, Kathryn Weinand, Anastasia Vedenko, Luis A Barrera and Martha L Bulyk. [Cell Systems, 5 (3):187-201.e7, (2017)](https://www.sciencedirect.com/science/article/pii/S2405471217302892)
  - **In-silico read normalization using set multicover optimization.** Dilip Durai and Marcel Schulz. [Bioinformatics, 34 (19): 3273-3280, (2018)](https://academic.oup.com/bioinformatics/article/34/19/3273/4975418)
  - **Repeat-aware evaluation of scaffolding tools.** Igor Mandric, Sergey Knyazev and Alex Zelikovsky. [Bioinformatics, 34 (15): 2530-2537, (2018)](https://academic.oup.com/bioinformatics/article/34/15/2530/4934936)
  - **Faucet: streaming de novo assembly graph construction.** Roye Rozov, Gil Goldshlager, Ron Shamir and Eran Halperin. [Bioinformatics, 34 (1): 147-154, (2018)](https://academic.oup.com/bioinformatics/article/34/1/147/4004871)

##### 2016

  - **Accurate self correction of errors in long reads using de Bruijn graphs.** Leena Salmela, Riku Walve, Eric Rivals and Esko Ukkonen. [Bioinformatics, 33 (6): 799–806](https://academic.oup.com/bioinformatics/article/33/6/799/2525585)
  - **Designing robust watermark barcodes for multiplex long-read sequencing.** Joaquín Ezpeleta, Flavia J. Krsticevic, Pilar Bulacio and Elizabeth Tapia. [Bioinformatics, 33 (6): 807-813](https://academic.oup.com/bioinformatics/article/33/6/807/2525586)
  - **COCACOLA: binning metagenomic contigs using sequence COmposition, read CoverAge, CO-alignment, and paired-end read LinkAge.** Yang Lu, Ting Chen, Jed Fuhrman and Fengzhu Sun. [Bioinformatics, 33 (6): 791-798](https://academic.oup.com/bioinformatics/article/33/6/791/2525584)

##### 2015

  - **PopIns: population-scale detection of novel sequence insertions.** Birte Kehr, Pall Melsted and Bjarni Halldorsson. [Bioinformatics,  32 (7): 961-967, (2016)](https://academic.oup.com/bioinformatics/article/32/7/961/2240308?searchresult=1)
  - **BMix: probabilistic modeling of occurring substitutions in PAR-CLIP data.** Monica Golumbeanu, Pejman Mohammadi and Niko Beerenwinkel. [Bioinformatics,  32 (7): 976-983, (2016)](https://academic.oup.com/bioinformatics/article/32/7/976/2288374?searchresult=1)
  - **SV-Bay: structural variant detection in cancer genomes using a Bayesian approach with correction for GC-content and read mappability.** Daria Iakovishina, Isabelle Janoueix-Lerosey, Emmanuel Barillot, Mireille Regnier and Valentina Boeva. [Bioinformatics,  32 (7): 984-992, (2016)](https://academic.oup.com/bioinformatics/article/32/7/984/2288622?searchresult=1)
  - **Inference of Markovian properties of molecular sequences from NGS data and applications to comparative genomics.** Jie Ren, Kai Song, Minghua Deng, Gesine Reinert, Chuck Cannon and Fengzhu Sun. [Bioinformatics,  32 (7): 993-1000, (2016)](https://academic.oup.com/bioinformatics/article/32/7/993/2240327?searchresult=1)
  - **Utilizing de Bruijn graph of metagenome assembly for metatranscriptome analysis.** Yuzhen Ye and Haixu Tang . [Bioinformatics,  32 (7): 1001-1008, (2016)](https://academic.oup.com/bioinformatics/article/32/7/1001/2288363?searchresult=1)
  - **OPTIMA: index-based map-to-sequence alignment in large eukaryotic genomes.** Davide Verzotto, Audrey S.M. Teo, Axel Hillmer and Niranjan Nagarajan. [GigaScience, 5 (1): s13742-016-0110-0, (2016)](https://academic.oup.com/gigascience/article/5/1/s13742-016-0110-0/2720967)
  - **Rank regularized RNA-seq.** Nicolas Bray and Lior Pachter.

##### 2014

  - **Assembly-free genome comparison based on next-generation sequencing reads and variable length patterns.** Matteo Comin and Michele Schimd. [BMC Bioinformatics 15 (Suppl 9) S1 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S1)
  - **SWAP-Assembler: scalable and efficient genome assembly towards thousands of cores.** Jintao Meng, Bingqiang Wang, Yanjie Wei, Shengzhong Feng and Pavan Balaji. [BMC Bioinformatics 15 (Suppl 9) S2 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S2)
  - **Piecing the puzzle together: a revisit to transcript reconstruction problem in RNA-seq.** Yan Huang, Yin Hu and Jinze Liu. [BMC Bioinformatics 15 (Suppl 9) S3 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S3)
  - **Near-optimal assembly for shotgun sequencing with noisy reads.** Ka-Kit Lam, Asif Khalak and David Tse. [BMC Bioinformatics 15 (Suppl 9) S4 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S4)
  - **On the complexity of Minimum Path Cover with Subpath Constraints for multi-assembly.** Romeo Rizzi, Alexandru I Tomescu and Veli Mäkinen. [BMC Bioinformatics 15 (Suppl 9) S5 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S5)
  - **BADGE: A novel Bayesian model for accurate abundance quantification and differential analysis of RNA-Seq data.** Jinghua Gu, Xiao Wang, Leena Halakivi-Clarke, Robert Clarke and Jianhua Xuan. [BMC Bioinformatics 15 (Suppl 9) S6 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S6)
  - **Fast lossless compression via cascading Bloom filters.** Roye Rozov, Ron Shamir and Eran Halperin. [BMC Bioinformatics 15 (Suppl 9) S7 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S7)
  - **Gene finding in metatranscriptomic sequences.** Wazim Mohammed Ismail, Yuzhen Ye and Haixu Tang. [BMC Bioinformatics 15 (Suppl 9) S8 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S8)
  - **ILP-based maximum likelihood genome scaffolding.** James Lindsay, Hamed Salooti, Ion Măndoiu and Alex Zelikovsky. [BMC Bioinformatics 15 (Suppl 9) S9 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S9)
  - **CASPER: context-aware scheme for paired-end reads from high-throughput amplicon sequencing.** Sunyoung Kwon, Byunghan Lee and Sungroh Yoon. [BMC Bioinformatics 15 (Suppl 9) S10 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S10)
  - **PuFFIN - a parameter-free method to build nucleosome maps from paired-end reads.** Anton Polishko, Evelien M Bunnik, Karine G Le Roch and Stefano Lonardi. [BMC Bioinformatics 15 (Suppl 9) S11 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S11)
  - **ARYANA: Aligning Reads by Yet Another Approach.** Milad Gholami, Aryan Arbabi, Ali Sharifi-Zarchi, Hamidreza Chitsaz and Mehdi Sadeghi. [BMC Bioinformatics 15 (Suppl 9) S12 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S12)
  - **Ensemble analysis of adaptive compressed genome sequencing strategies.** Zeinab Taghavi. [BMC Bioinformatics 15 (Suppl 9) S13 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S13)
  - **A better sequence-read simulator program for metagenomics.** Stephen Johnson, Brett Trost, Jeffrey R Long, Vanessa Pittet and Anthony Kusalik. [BMC Bioinformatics 15 (Suppl 9) S14 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S14)
  - **ProbeAlign: incorporating high-throughput sequencing-based structure probing information into ncRNA homology search.** Ping Ge, Cuncong Zhong and Shaojie Zhang. [BMC Bioinformatics 15 (Suppl 9) S15 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S15)
  - **Detecting epigenetic motifs in low coverage and metagenomics settings.** Noam D Beckmann, Sashank Karri, Gang Fang and Ali Bashir. [BMC Bioinformatics 15 (Suppl 9) S16 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S16)


##### 2013

  - **Discovering motifs that induce sequencing errors.** Manuel Allhoff, Alexander Schönhuth, Marcel Martin, Ivan G Costa, Sven Rahmann and Tobias Marschall. [BMC Bioinformatics 14 (Suppl 5) S1 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S1)
  - **metaBEETL: high-throughput analysis of heterogeneous microbial populations from shotgun DNA sequences.** Christina Ander, Ole B Schulz-Trieglaff, Jens Stoye and Anthony J Cox. [BMC Bioinformatics 14 (Suppl 5) S2 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S2)
  - **Joint genotype inference with germline and somatic mutations.** Eric Bareke, Virginie Saillour, Jean-François Spinella, Ramon Vidal, Jasmine Healy, Daniel Sinnett and Miklós Csűrös. [BMC Bioinformatics 14 (Suppl 5) S3 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S3)
  - **Discovering and mapping chromatin states using a tree hidden Markov model.** Jacob Biesinger, Yuanfeng Wang and Xiaohui Xie. [BMC Bioinformatics 14 (Suppl 5) S4 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S4)
  - **Prioritization of candidate disease genes by topological similarity between disease and protein diffusion profiles.** Jie Zhu, Yufang Qin, Taigang Liu, Jun Wang and Xiaoqi Zheng. [BMC Bioinformatics 14 (Suppl 5) S5 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S5)
  - **Bellerophon: a hybrid method for detecting interchromo-somal rearrangements at base pair resolution using next-generation sequencing data.** Matthew Hayes and Jing Li. [BMC Bioinformatics 14 (Suppl 5) S6 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S6)
  - **De Bruijn superwalk with multiplicities problem is NP-hard.** Evgeny Kapun and Fedor Tsarev. [BMC Bioinformatics 14 (Suppl 5) S7 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S7)
  - **A random-permutations-based approach to fast read alignment.** Roy Lederman. [BMC Bioinformatics 14 (Suppl 5) S8 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S8)
  - **Assembling contigs in draft genomes using reversals and block-interchanges.** Chi-Long Li, Kun-Tze Chen and Chin Lung Lu. [BMC Bioinformatics 14 (Suppl 5) S9 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S9)
  - **An optimized algorithm for detecting and annotating regional differential methylation.** Sheng Li, Francine E Garrett-Bakelman, Altuna Akalin, Paul Zumbo, Ross Levine, Bik L To, Ian D Lewis, Anna L Brown, Richard J D'Andrea, Ari Melnick and Christopher E Mason. [BMC Bioinformatics 14 (Suppl 5) S10 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S10)
  - **A mixture model for expression deconvolution from RNA-seq in heterogeneous tissues.** Yi Li and Xiaohui Xie. [BMC Bioinformatics 14 (Suppl 5) S11 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S11)
  - **Gene prediction in metagenomic fragments based on the SVM algorithm.** Yongchu Liu, Jiangtao Guo, Gangqing Hu and Huaiqiu Zhu. [BMC Bioinformatics 14 (Suppl 5) S12 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S12)
  - **Evaluating genome architecture of a complex region via generalized bipartite matching.** Christine Lo, Sangwoo Kim, Shay Zakov and Vineet Bafna. [BMC Bioinformatics 14 (Suppl 5) S13 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S13)
  - **CLASS: constrained transcript assembly of RNA-seq reads.** Li Song and Liliana Florea. [BMC Bioinformatics 14 (Suppl 5) S14 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S14)
  - **A novel min-cost flow method for estimating transcript expression with RNA-Seq.** Alexandru I Tomescu, Anna Kuosmanen, Romeo Rizzi and Veli Mäkinen. [BMC Bioinformatics 14 (Suppl 5) S15 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S15)
  - **Gene set enrichment analysis of RNA-Seq data: integrating differential expression and splicing.** Xi Wang and Murray J Cairns. [BMC Bioinformatics 14 (Suppl 5) S16 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S16)
  - **De novo inference of stratification and local admixture in sequencing studies.** Yu Zhang. [BMC Bioinformatics 14 (Suppl 5) S17 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S17)
  - **Optimal assembly for high throughput shotgun sequencing.** Guy Bresler, Ma'ayan Bresler and David Tse. [BMC Bioinformatics 14 (Suppl 5) S18 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S18)


##### 2012

  - **Exploiting sparseness in *de novo* genome assembly.** Chengxi Ye, Zhanshan Sam Ma, Charles H Cannon, Mihai Pop and Douglas W Yu. [BMC Bioinformatics 13 (Suppl 6), S1 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S1).
  - **MGMR: leveraging RNA-Seq population data to optimize expression estimation.** Roye Rozov, Eran Halperin and Ron Shamir. [BMC Bioinformatics 13 (Suppl 6), S2 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S2).
  - **Haplotype reconstruction using perfect phylogeny and sequence data.** Anatoly Efros and Eran Halperin. [BMC Bioinformatics 13 (Suppl 6), S3 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S3).
  - **Biases in read coverage demonstrated by interlaboratory and interplatform comparison of 117 mRNA and genome sequencing experiments.** Ekaterina E Khrameeva and Mikhail S Gelfand. [BMC Bioinformatics 13 (Suppl 6), S4 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S4).
  - **KIS SPLICE: de-novo calling alternative splicing events from RNA-seq data.** Gustavo AT Sacomoto, Janice Kielbassa, Rayan Chikhi, Raluca Uricaru, Pavlos Antoniou, Marie-France Sagot,   Pierre Peterlongo and Vincent Lacroix. [BMC Bioinformatics 13 (Suppl 6), S5 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S5).
  - **An improved approach for accurate and efficient calling of structural variations with low-coverage sequence data.** Jin Zhang, Jiayin Wang and Yufeng Wu. [BMC Bioinformatics 13 (Suppl 6), S6 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S6).
  - **PAIR: polymorphic *Alu* insertion recognition.** Jón Ingi Sveinbjörnsson and Bjarni V Halldórsson. [BMC Bioinformatics 13 (Suppl 6), S7 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S7).
  - **High-resolution genetic mapping with pooled sequencing.** Matthew D Edwards and David K Gifford. [BMC Bioinformatics 13 (Suppl 6), S8 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S8).
  - **A context-based approach to identify the most likely mapping for RNA-seq experiments.** Thomas Bonfert, Gergely Csaba, Ralf Zimmer and Caroline C Friedel. [BMC Bioinformatics 13 (Suppl 6), S9 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S9).
  - **Reconstructing cancer genomes from paired-end sequencing data.** Layla Oesper, Anna Ritz, Sarah J Aerni, Ryan Drebin and Benjamin J Raphael. [BMC Bioinformatics 13 (Suppl 6), S10 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S10).
  - **Challenges in estimating percent inclusion of alternatively spliced junctions from RNA-seq data.** Boyko Kakaradov, Hui Yuan Xiong, Leo J Lee, Nebojsa Jojic and Brendan J Frey. [BMC Bioinformatics 13 (Suppl 6), S11 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S11).

##### 2011

  - **Inferring viral spectrum from 454 pyrosequencing reads.** Irina Astrovskaya, Bassam Tork, Serghei Mangul, Kelly Westbrooks, Ion Mandoiu, Peter Balfe and Alex Zelikovsky. [BMC Bioinformatics 12 (Suppl 6), S1 (2011)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-S6-S1).
  - **Genotyping common and rare variation using overlapping pool sequencing.** Dan He, Noah Zaitlen, Bogdan Pasaniuc, Eleazar Eskin and Eran Halperin. [BMC Bioinformatics 12 (Suppl 6), S2 (2011)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-S6-S2).
  - **Assembly of non-unique insertion content using next-generation sequencing.** Nathaniel Parrish, Farhad Hormozdiari and Eleazar Eskin. [BMC Bioinformatics 12 (Suppl 6), S3 (2011)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-S6-S3).
  - **A Hidden Markov Model for copy number variant prediction from whole genome resequencing data.** Yufeng Shen, Yiwei Gu and Itsik Pe'Er. [BMC Bioinformatics 12 (Suppl 6), S4 (2011)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-S6-S4).


##### [Short Talks Selected from Abstracts and Posters](/abstracts)

