---
layout: post
title: Proceedings of RECOMB-Seq
permalink: /proceedings/
isStaticPost: true
---

#### Full Proceedings Papers

##### 2025

  - **Alignment-free detection of differences between sequencing data sets.** Alessia Petescia, Luca Denti, Askar Gafurov, Viktória Hodorová, Jozef Nosek, Broňa Brejová and Tomas Vinar. 
  - **Tile-X: A vertex reordering approach for scalable long read assembly.** Oieswarya Bhowmik and Ananth Kalyanaraman. [bioRxiv 2025.04.21.649853, (2025)](https://www.biorxiv.org/content/10.1101/2025.04.21.649853v1.full)
  - **Decoding the causal drivers of spatial cellular topology.** Prannav Shankar, Huan Liang, Uthsav Chitra and Rohit Singh. [bioRxiv 2025.03.19.644241, (2025)](https://www.biorxiv.org/content/10.1101/2025.03.19.644241v2)
  - **Efficient trace reconstruction in DNA storage systems using Bidirectional Beam Search.** Zhenhao Gu, Hongyi Xin, Puru Sharma, Gary Yipeng Goh, Limsoon Wong and Niranjan Nagarajan. [bioRxiv 2025.04.16.644694, (2025)](https://www.biorxiv.org/content/10.1101/2025.04.16.644694v1)
  - **Decoupling Hi-C matrices with non-negative matrix factorization to reveal assorted TADs.** Zhao Ling, Shi Ying Li, Jingwan Wang, Xikang Feng, Bowen Tan and Shuai Cheng Li. 
  - **Run-length compressed metagenomic read classification with SMEM-finding and tagging.** Lore Depuydt, Omar Y. Ahmed, Jan Fostier, Ben Langmead and Travis Gagie. [bioRxiv 2025.02.25.640119, (2025)](https://www.biorxiv.org/content/10.1101/2025.02.25.640119v2)

##### 2023

  - **TreeTerminus - creating transcript trees using inferential replicate counts.** Noor Pratap Singh, Michael I Love and Rob Patro. [iScience, 26 (6): 106961, (2023)](https://www.sciencedirect.com/science/article/pii/S2589004223010386)
  - **DandD: efficient measurement of sequence growth and similarity.** Jessica Bonnie, Omar Ahmed and Ben Langmead. [iScience, 27 (3): 109054, (2024)](https://www.sciencedirect.com/science/article/pii/S258900422400275X)
  - **ADEPT: autoencoder with differentially expressed genes and imputation for a robust spatial transcriptomics clusterin.** Yunfei Hu, Yuying Zhao, Curtis Schunk, Yingxiang Ma, Tyler Derr and Xin Maizie Zhou. [iScience, 26 (6): 106792, (2023)](https://www.sciencedirect.com/science/article/pii/S2589004223008696)
  - **aKmerBroom: ancient oral DNA decontamination using Bloom filters on k-mer sets.** Camila Duitama González, Riccardo Vicedomini, Hugues Richard and Rayan Chikhi. [iScience, 26 (11): 108057, (2023)](https://www.sciencedirect.com/science/article/pii/S258900422302134X)
  - **Masked superstrings as a unified framework for textual k-mer set representations.** Ondrej Sladky, Pavel Vesely and Karel Brinda. [bioRxiv 2023.02.01.526717, (2023)](https://www.biorxiv.org/content/10.1101/2023.02.01.526717v1)
  - **WGT: tools and algorithms for recognizing, visualizing and generating Wheeler graphs.** Kuan-Hao Chao, Pei-Wei Chen, Sanjit A. Seshia and Ben Langmead. [iScience, 26 (8): 107402, (2023)](https://www.sciencedirect.com/science/article/pii/S2589004223014797)

##### 2022

  - **Sequence-based pangenomic core detection.** Tizian Schulz, Roland Wittler and Jens Stoye. [iScience, 25 (6): 104413, (2022)](https://www.sciencedirect.com/science/article/pii/S2589004222006848)
  - **scTagger: Fast and accurate matching of cellular barcodes across short- and long-reads of single-cell RNA-seq experiments.** Ghazal Ebrahimi, Baraa Orabi, Meghan Robinson, Cedric Chauve, Ryan Flannigan and Faraz Hach. [iScience, 25 (7): 104530, (2022)](https://www.sciencedirect.com/science/article/pii/S258900422200801X)
  - **Genetic polyploid phasing from low-depth progeny samples.** Sven Schrinner, Rebecca Serra Mari, Richard Finkers, Paul Arens, Björn Usadel, Tobias Marschall and Gunnar W. Klau. [iScience, 25 (6): 104461, (2022)](https://www.sciencedirect.com/science/article/pii/S2589004222007325)
  - **BOA: a partitioned view of genome assembly .** Priyanka Ghosh, Xiaojing An, Patrick Keppler, Sureyya Emre Kurt, Umit Catalyurek, Sriram Krishnamoorthy, Ponnuswamy Sadayappan, Aravind Sukumaran Rajam and Ananth Kalyanaraman. [iScience, 25 (11): 105273, (2022)](https://www.sciencedirect.com/science/article/pii/S2589004222015450)
  - **Mapping-friendly sequence reductions: going beyond homopolymer compression.** Luc Blassel, Paul Medvedev and Rayan Chikhi. [iScience, 25 (11): 105305, (2022)](https://www.sciencedirect.com/science/article/pii/S2589004222015772)
  - **BinSPreader: refine binning results for fuller MAG reconstruction .** Ivan Tolstoganov, Yury Kamenev, Roman Kruglikov, Sofia Ochkalova and Anton Korobeynikov. [iScience, 25 (8): 104770, (2022)](https://www.sciencedirect.com/science/article/pii/S2589004222010422)

##### 2021

  - **Dynamic partitioning of search patterns for approximate pattern matching using search schemes.** Luca Renders, Kathleen Marchal and Jan Fostier. [iScience, 24 (7): 102687, (2021)](https://www.cell.com/iscience/fulltext/S2589-0042(21)00655-6)
  - **Linearization of genome sequence graphs revisited.** Anna Lisiecka and Norbert Dojer. [iScience, 24 (7): 102755, (2021)](https://www.cell.com/iscience/fulltext/S2589-0042(21)00723-9)
  - **Pan-genomic matching statistics for targeted nanopore sequencing.** Omar Ahmed, Massimiliano Rossi, Sam Kovaka, Michael Schatz, Travis Gagie, Christina Boucher and Ben Langmead. [iScience, 24 (6): 102696, (2021)](https://www.cell.com/iscience/fulltext/S2589-0042(21)00664-7)
  - **A molecular taxonomy of tumors independent of tissue-of-origin.** Peter Nguyen, Simon Coetzee, Daniel Lakeland and Dennis Hazelett . [iScience, 24 (10): 103084, (2021)](https://www.cell.com/iscience/fulltext/S2589-0042(21)01052-X)
  - **A reference-free approach for cell type classification with scRNA-seq.** Qi Sun, Yifan Peng and Jinze Liu. [iScience, 24 (8): 102855, (2021)](https://www.cell.com/iscience/fulltext/S2589-0042(21)00823-3)

##### 2018

  - **Full-length de novo viral quasispecies assembly through variation graph construction.** Jasmijn Baaijens, Bastiaan van der Roest, Johannes Koster, Leen Stougie and Alexander Schoenhuth. [Bioinformatics, 35 (24): 5086-5094, (2019)](https://academic.oup.com/bioinformatics/article/35/24/5086/5506652)
  - **Alignment-free clustering of barcode (UMI) tagged DNA molecules.** Baraa Orabi, Emre Erhan, Brian McConeghy, Stanislav V. Volik, Stephane Le Bihan, Collin C. Collins, Cedric Chauve and Faraz Hach. [Bioinformatics, 35 (11): 1829-1836, (2019)](https://academic.oup.com/bioinformatics/article/35/11/1829/5142725)
  - **Toward perfect reads: self-correction of short reads via mapping on de Bruijn graphs.** Antoine Limasset, Jean-François Flot and Pierre Peterlongo. [Bioinformatics, 36 (5): 1374-1381, (2020)](https://academic.oup.com/bioinformatics/article/36/5/1374/5345559)
  - **TNER: a novel background error suppression method for mutation detection in circulating tumor DNA.** Shibing Deng, Maruja Lira, Donghui Huang, Kai Wang, Crystal Valdez, Jennifer Kinong, Paul Rejto, Jadwiga Bienkowska, James Hardwick and Tao Xie. [BMC Bioinformatics, 18: 387, (2018)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2428-3)
  - **Optimum search schemes for approximate string matching using bidirectional FM-Index.** Kiavash Kianfar, Christopher Pockrandt, Bahman Torkamandi, Haochen Luo and Knut Reinert. [bioRxiv 301085, (2018)](https://www.biorxiv.org/content/10.1101/301085v2)
  - **Resistome SNP calling via read colored de Bruijn graphs .** Bahar Alipanahi, Martin Muggli, Musa Jundi, Noelle Noyes and Christina Boucher. [Bioinformatics, 36 (22-23): 5275-5281, (2021)](https://academic.oup.com/bioinformatics/article/36/22-23/5275/5734643)
  - **Deconvoluting the diversity of within-host pathogen strains in a Multi-Locus Sequence Typing framework.** Guo Liang Gan, Elijah Willie, Cedric Chauve and Leonid Chindelevitch. [BMC Bioinformatics, 20: 637, (2019)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3204-8)

##### 2017

  - **QUENTIN: accurate reconstruction of disease transmissions from viral quasispecies genomic data.** Pavel Skums, Alex Zelikovsky, Sergey Knyazev, Igor Mandric, Zoya Dimitrova, Sumathi Ramachandran, David Stiven Campo Rendon, Leonid Bunimovich, Elizabeth Costenbader, Connie Sexton, Siobhan O'Connor, Guo-Liang Xia and Yury Khudyakov. [Bioinformatics, 34 (1): 163-170, (2018)](https://academic.oup.com/bioinformatics/article/34/1/163/3870483)
  - **An accurate algorithm for the detection of DNA fragments from dilution pool sequencing experiments.** Vikas Bansal. [Bioinformatics, 34 (1): 155-162, (2018)](https://academic.oup.com/bioinformatics/article/34/1/155/3952670)
  - **A novel data structure to support ultra-fast taxonomic classification of metagenomic sequences with k-mer signatures.** Xinan Liu, Ye Yu, Jinpeng Liu, Chen Qian and Jinze Liu. [Bioinformatics, 34 (1): 171-178, (2018)](https://academic.oup.com/bioinformatics/article/34/1/171/3931858)
  - **DIMM-SC: A Dirichlet mixture model for clustering droplet-based single cell transcriptomic data.** Zhe Sun, Ting Wang, Ke Deng, Xiao-Feng Wang, Robert Lafyatis, Ying Ding, Ming Hu and Wei Chen. [Bioinformatics, 34 (1): 139-146, (2018)](https://academic.oup.com/bioinformatics/article/34/1/139/4060554)
  - **Transcription factor-8mer glossary with GENRE genomic background enables precise identification of lineage-specific co-regulators.** Luca Mariani, Kathryn Weinand, Anastasia Vedenko, Luis A Barrera and Martha L Bulyk. [Cell Systems, 5 (3):187-201.e7, (2017)](https://www.sciencedirect.com/science/article/pii/S2405471217302892)
  - **In-silico read normalization using set multicover optimization.** Dilip Durai and Marcel Schulz. [Bioinformatics, 34 (19): 3273-3280, (2018)](https://academic.oup.com/bioinformatics/article/34/19/3273/4975418)
  - **Repeat-aware evaluation of scaffolding tools.** Igor Mandric, Sergey Knyazev and Alex Zelikovsky. [Bioinformatics, 34 (15): 2530-2537, (2018)](https://academic.oup.com/bioinformatics/article/34/15/2530/4934936)
  - **Faucet: streaming de novo assembly graph construction.** Roye Rozov, Gil Goldshlager, Ron Shamir and Eran Halperin. [Bioinformatics, 34 (1): 147-154, (2018)](https://academic.oup.com/bioinformatics/article/34/1/147/4004871)

##### 2016

  - **Accurate self correction of errors in long reads using de Bruijn graphs.** Leena Salmela, Riku Walve, Eric Rivals and Esko Ukkonen. [Bioinformatics, 33 (6): 799–806](https://academic.oup.com/bioinformatics/article/33/6/799/2525585)
  - **Designing robust watermark barcodes for multiplex long-read sequencing.** Joaquín Ezpeleta, Flavia J. Krsticevic, Pilar Bulacio and Elizabeth Tapia. [Bioinformatics, 33 (6): 807-813](https://academic.oup.com/bioinformatics/article/33/6/807/2525586)
  - **COCACOLA: binning metagenomic contigs using sequence COmposition, read CoverAge, CO-alignment, and paired-end read LinkAge.** Yang Lu, Ting Chen, Jed Fuhrman and Fengzhu Sun. [Bioinformatics, 33 (6): 791-798](https://academic.oup.com/bioinformatics/article/33/6/791/2525584)

##### 2015

  - **PopIns: population-scale detection of novel sequence insertions.** Birte Kehr, Pall Melsted and Bjarni Halldorsson. [Bioinformatics,  32 (7): 961-967, (2016)](https://academic.oup.com/bioinformatics/article/32/7/961/2240308?searchresult=1)
  - **BMix: probabilistic modeling of occurring substitutions in PAR-CLIP data.** Monica Golumbeanu, Pejman Mohammadi and Niko Beerenwinkel. [Bioinformatics,  32 (7): 976-983, (2016)](https://academic.oup.com/bioinformatics/article/32/7/976/2288374?searchresult=1)
  - **SV-Bay: structural variant detection in cancer genomes using a Bayesian approach with correction for GC-content and read mappability.** Daria Iakovishina, Isabelle Janoueix-Lerosey, Emmanuel Barillot, Mireille Regnier and Valentina Boeva. [Bioinformatics,  32 (7): 984-992, (2016)](https://academic.oup.com/bioinformatics/article/32/7/984/2288622?searchresult=1)
  - **Inference of Markovian properties of molecular sequences from NGS data and applications to comparative genomics.** Jie Ren, Kai Song, Minghua Deng, Gesine Reinert, Chuck Cannon and Fengzhu Sun. [Bioinformatics,  32 (7): 993-1000, (2016)](https://academic.oup.com/bioinformatics/article/32/7/993/2240327?searchresult=1)
  - **Utilizing de Bruijn graph of metagenome assembly for metatranscriptome analysis.** Yuzhen Ye and Haixu Tang . [Bioinformatics,  32 (7): 1001-1008, (2016)](https://academic.oup.com/bioinformatics/article/32/7/1001/2288363?searchresult=1)
  - **OPTIMA: index-based map-to-sequence alignment in large eukaryotic genomes.** Davide Verzotto, Audrey S.M. Teo, Axel Hillmer and Niranjan Nagarajan. [GigaScience, 5 (1): s13742-016-0110-0, (2016)](https://academic.oup.com/gigascience/article/5/1/s13742-016-0110-0/2720967)
  - **Rank regularized RNA-seq.** Nicolas Bray and Lior Pachter.

##### 2014

  - **Assembly-free genome comparison based on next-generation sequencing reads and variable length patterns.** Matteo Comin and Michele Schimd. [BMC Bioinformatics 15 (Suppl 9) S1 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S1)
  - **SWAP-Assembler: scalable and efficient genome assembly towards thousands of cores.** Jintao Meng, Bingqiang Wang, Yanjie Wei, Shengzhong Feng and Pavan Balaji. [BMC Bioinformatics 15 (Suppl 9) S2 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S2)
  - **Piecing the puzzle together: a revisit to transcript reconstruction problem in RNA-seq.** Yan Huang, Yin Hu and Jinze Liu. [BMC Bioinformatics 15 (Suppl 9) S3 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S3)
  - **Near-optimal assembly for shotgun sequencing with noisy reads.** Ka-Kit Lam, Asif Khalak and David Tse. [BMC Bioinformatics 15 (Suppl 9) S4 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S4)
  - **On the complexity of Minimum Path Cover with Subpath Constraints for multi-assembly.** Romeo Rizzi, Alexandru I Tomescu and Veli Mäkinen. [BMC Bioinformatics 15 (Suppl 9) S5 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S5)
  - **BADGE: A novel Bayesian model for accurate abundance quantification and differential analysis of RNA-Seq data.** Jinghua Gu, Xiao Wang, Leena Halakivi-Clarke, Robert Clarke and Jianhua Xuan. [BMC Bioinformatics 15 (Suppl 9) S6 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S6)
  - **Fast lossless compression via cascading Bloom filters.** Roye Rozov, Ron Shamir and Eran Halperin. [BMC Bioinformatics 15 (Suppl 9) S7 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S7)
  - **Gene finding in metatranscriptomic sequences.** Wazim Mohammed Ismail, Yuzhen Ye and Haixu Tang. [BMC Bioinformatics 15 (Suppl 9) S8 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S8)
  - **ILP-based maximum likelihood genome scaffolding.** James Lindsay, Hamed Salooti, Ion Măndoiu and Alex Zelikovsky. [BMC Bioinformatics 15 (Suppl 9) S9 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S9)
  - **CASPER: context-aware scheme for paired-end reads from high-throughput amplicon sequencing.** Sunyoung Kwon, Byunghan Lee and Sungroh Yoon. [BMC Bioinformatics 15 (Suppl 9) S10 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S10)
  - **PuFFIN - a parameter-free method to build nucleosome maps from paired-end reads.** Anton Polishko, Evelien M Bunnik, Karine G Le Roch and Stefano Lonardi. [BMC Bioinformatics 15 (Suppl 9) S11 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S11)
  - **ARYANA: Aligning Reads by Yet Another Approach.** Milad Gholami, Aryan Arbabi, Ali Sharifi-Zarchi, Hamidreza Chitsaz and Mehdi Sadeghi. [BMC Bioinformatics 15 (Suppl 9) S12 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S12)
  - **Ensemble analysis of adaptive compressed genome sequencing strategies.** Zeinab Taghavi. [BMC Bioinformatics 15 (Suppl 9) S13 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S13)
  - **A better sequence-read simulator program for metagenomics.** Stephen Johnson, Brett Trost, Jeffrey R Long, Vanessa Pittet and Anthony Kusalik. [BMC Bioinformatics 15 (Suppl 9) S14 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S14)
  - **ProbeAlign: incorporating high-throughput sequencing-based structure probing information into ncRNA homology search.** Ping Ge, Cuncong Zhong and Shaojie Zhang. [BMC Bioinformatics 15 (Suppl 9) S15 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S15)
  - **Detecting epigenetic motifs in low coverage and metagenomics settings.** Noam D Beckmann, Sashank Karri, Gang Fang and Ali Bashir. [BMC Bioinformatics 15 (Suppl 9) S16 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-S9-S16)


##### 2013

  - **Discovering motifs that induce sequencing errors.** Manuel Allhoff, Alexander Schönhuth, Marcel Martin, Ivan G Costa, Sven Rahmann and Tobias Marschall. [BMC Bioinformatics 14 (Suppl 5) S1 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S1)
  - **metaBEETL: high-throughput analysis of heterogeneous microbial populations from shotgun DNA sequences.** Christina Ander, Ole B Schulz-Trieglaff, Jens Stoye and Anthony J Cox. [BMC Bioinformatics 14 (Suppl 5) S2 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S2)
  - **Joint genotype inference with germline and somatic mutations.** Eric Bareke, Virginie Saillour, Jean-François Spinella, Ramon Vidal, Jasmine Healy, Daniel Sinnett and Miklós Csűrös. [BMC Bioinformatics 14 (Suppl 5) S3 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S3)
  - **Discovering and mapping chromatin states using a tree hidden Markov model.** Jacob Biesinger, Yuanfeng Wang and Xiaohui Xie. [BMC Bioinformatics 14 (Suppl 5) S4 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S4)
  - **Prioritization of candidate disease genes by topological similarity between disease and protein diffusion profiles.** Jie Zhu, Yufang Qin, Taigang Liu, Jun Wang and Xiaoqi Zheng. [BMC Bioinformatics 14 (Suppl 5) S5 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S5)
  - **Bellerophon: a hybrid method for detecting interchromo-somal rearrangements at base pair resolution using next-generation sequencing data.** Matthew Hayes and Jing Li. [BMC Bioinformatics 14 (Suppl 5) S6 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S6)
  - **De Bruijn superwalk with multiplicities problem is NP-hard.** Evgeny Kapun and Fedor Tsarev. [BMC Bioinformatics 14 (Suppl 5) S7 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S7)
  - **A random-permutations-based approach to fast read alignment.** Roy Lederman. [BMC Bioinformatics 14 (Suppl 5) S8 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S8)
  - **Assembling contigs in draft genomes using reversals and block-interchanges.** Chi-Long Li, Kun-Tze Chen and Chin Lung Lu. [BMC Bioinformatics 14 (Suppl 5) S9 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S9)
  - **An optimized algorithm for detecting and annotating regional differential methylation.** Sheng Li, Francine E Garrett-Bakelman, Altuna Akalin, Paul Zumbo, Ross Levine, Bik L To, Ian D Lewis, Anna L Brown, Richard J D'Andrea, Ari Melnick and Christopher E Mason. [BMC Bioinformatics 14 (Suppl 5) S10 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S10)
  - **A mixture model for expression deconvolution from RNA-seq in heterogeneous tissues.** Yi Li and Xiaohui Xie. [BMC Bioinformatics 14 (Suppl 5) S11 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S11)
  - **Gene prediction in metagenomic fragments based on the SVM algorithm.** Yongchu Liu, Jiangtao Guo, Gangqing Hu and Huaiqiu Zhu. [BMC Bioinformatics 14 (Suppl 5) S12 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S12)
  - **Evaluating genome architecture of a complex region via generalized bipartite matching.** Christine Lo, Sangwoo Kim, Shay Zakov and Vineet Bafna. [BMC Bioinformatics 14 (Suppl 5) S13 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S13)
  - **CLASS: constrained transcript assembly of RNA-seq reads.** Li Song and Liliana Florea. [BMC Bioinformatics 14 (Suppl 5) S14 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S14)
  - **A novel min-cost flow method for estimating transcript expression with RNA-Seq.** Alexandru I Tomescu, Anna Kuosmanen, Romeo Rizzi and Veli Mäkinen. [BMC Bioinformatics 14 (Suppl 5) S15 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S15)
  - **Gene set enrichment analysis of RNA-Seq data: integrating differential expression and splicing.** Xi Wang and Murray J Cairns. [BMC Bioinformatics 14 (Suppl 5) S16 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S16)
  - **De novo inference of stratification and local admixture in sequencing studies.** Yu Zhang. [BMC Bioinformatics 14 (Suppl 5) S17 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S17)
  - **Optimal assembly for high throughput shotgun sequencing.** Guy Bresler, Ma'ayan Bresler and David Tse. [BMC Bioinformatics 14 (Suppl 5) S18 (2013)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-S5-S18)


##### 2012

  - **Exploiting sparseness in *de novo* genome assembly.** Chengxi Ye, Zhanshan Sam Ma, Charles H Cannon, Mihai Pop and Douglas W Yu. [BMC Bioinformatics 13 (Suppl 6), S1 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S1).
  - **MGMR: leveraging RNA-Seq population data to optimize expression estimation.** Roye Rozov, Eran Halperin and Ron Shamir. [BMC Bioinformatics 13 (Suppl 6), S2 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S2).
  - **Haplotype reconstruction using perfect phylogeny and sequence data.** Anatoly Efros and Eran Halperin. [BMC Bioinformatics 13 (Suppl 6), S3 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S3).
  - **Biases in read coverage demonstrated by interlaboratory and interplatform comparison of 117 mRNA and genome sequencing experiments.** Ekaterina E Khrameeva and Mikhail S Gelfand. [BMC Bioinformatics 13 (Suppl 6), S4 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S4).
  - **KIS SPLICE: de-novo calling alternative splicing events from RNA-seq data.** Gustavo AT Sacomoto, Janice Kielbassa, Rayan Chikhi, Raluca Uricaru, Pavlos Antoniou, Marie-France Sagot,   Pierre Peterlongo and Vincent Lacroix. [BMC Bioinformatics 13 (Suppl 6), S5 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S5).
  - **An improved approach for accurate and efficient calling of structural variations with low-coverage sequence data.** Jin Zhang, Jiayin Wang and Yufeng Wu. [BMC Bioinformatics 13 (Suppl 6), S6 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S6).
  - **PAIR: polymorphic *Alu* insertion recognition.** Jón Ingi Sveinbjörnsson and Bjarni V Halldórsson. [BMC Bioinformatics 13 (Suppl 6), S7 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S7).
  - **High-resolution genetic mapping with pooled sequencing.** Matthew D Edwards and David K Gifford. [BMC Bioinformatics 13 (Suppl 6), S8 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S8).
  - **A context-based approach to identify the most likely mapping for RNA-seq experiments.** Thomas Bonfert, Gergely Csaba, Ralf Zimmer and Caroline C Friedel. [BMC Bioinformatics 13 (Suppl 6), S9 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S9).
  - **Reconstructing cancer genomes from paired-end sequencing data.** Layla Oesper, Anna Ritz, Sarah J Aerni, Ryan Drebin and Benjamin J Raphael. [BMC Bioinformatics 13 (Suppl 6), S10 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S10).
  - **Challenges in estimating percent inclusion of alternatively spliced junctions from RNA-seq data.** Boyko Kakaradov, Hui Yuan Xiong, Leo J Lee, Nebojsa Jojic and Brendan J Frey. [BMC Bioinformatics 13 (Suppl 6), S11 (2012)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S6-S11).

##### 2011

  - **Inferring viral spectrum from 454 pyrosequencing reads.** Irina Astrovskaya, Bassam Tork, Serghei Mangul, Kelly Westbrooks, Ion Mandoiu, Peter Balfe and Alex Zelikovsky. [BMC Bioinformatics 12 (Suppl 6), S1 (2011)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-S6-S1).
  - **Genotyping common and rare variation using overlapping pool sequencing.** Dan He, Noah Zaitlen, Bogdan Pasaniuc, Eleazar Eskin and Eran Halperin. [BMC Bioinformatics 12 (Suppl 6), S2 (2011)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-S6-S2).
  - **Assembly of non-unique insertion content using next-generation sequencing.** Nathaniel Parrish, Farhad Hormozdiari and Eleazar Eskin. [BMC Bioinformatics 12 (Suppl 6), S3 (2011)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-S6-S3).
  - **A Hidden Markov Model for copy number variant prediction from whole genome resequencing data.** Yufeng Shen, Yiwei Gu and Itsik Pe'Er. [BMC Bioinformatics 12 (Suppl 6), S4 (2011)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-S6-S4).

#### Overlay Papers (full papers published elsewhere)

##### 2025

  - **De novo clustering of extensive long-read transcriptome datasets with isONclust3.** Alexander Petri and Kristoffer Sahlin.
  - **Multi-context seeds enable fast and high-accuracy read mapping.** Ivan Tolstoganov, Marcel Martin and Kristoffer Sahlin.
  - **GPU-accelerated homology search with MMseqs2.** Felix Kallenborn, Alejandro Chacon, Christian Hundt, Hassan Sirelkhatim, Kieran Didi, Sooyoung Cha, Christian Dallago, Milot Mirdita, Bertil Schmidt and Martin Steinegger.
  - **DeepMM: identify and correct metagenome bisassemblies with deep learning.** Yi Ding, Jin Xiao, Bohao Zou and Lu Zhang.
  - **On the coverage required for diploid genome assembly.** Daanish Mahajan, Chirag Jain and Navin Kashyap.
  - **Pangenome graph augmentation from unassembled long reads.** Luca Denti, Paola Bonizzoni, Brona Brejova, Rayan Chikhi, Thomas Krannich, Tomas Vinar and Fereydoun Hormozdiari.

##### 2023

  - **Exploring high-quality microbial genomes by assembly of linked-reads with high barcode specificity using deep learning.** Zhenmiao Zhang, Hongbo Wang, Chao Yang, Yufen Huang, Zhen Yue, Yang Chen, Lijuan Han, Aiping Lyu, Xiaodong Fang and Lu Zhang.
  - **A safety framework for flow decomposition problems via integer linear programming.** Fernando Hugo Cunha Dias, Manuel Caceres, Lucia Williams, Brendan Mumey and Alexandru I. Tomescu.
  - **Coverage-preserving sparsification of overlap graphs for long-read assembly.** Chirag Jain.
  - **Predicting transcription factor activity using prior biological information.** Joseph Estabrook, William Yashar, Hannah Holly, Julia Somers, Olga Nikolova, Ozgun Babur, Theodore Braun and Emek Demir.
  - **On minimizers and convolutional filters: a partial justification for the effectiveness of CNNs in categorical sequence analysis.** Yun William Yu.
  - **TargetCall: eliminating the wasted computation in basecalling via pre-basecalling filtering.** Meryem Banu Cavlak, Gagandeep Singh, Mohammed Alser, Can Firtina, Joël Lindegger, Mohammad Sadrosadati, Nika Mansouri Ghiasi, Can Alkan and Onur Mutlu.
  - **An intrinsically interpretable neural network architecture for sequence to function learning.** Ali Tugrul Balci, Mark Maher Ebeid, Panayiotis Benos, Dennis Kostka and Maria Chikina.
  - **Scrooge: a fast and memory-frugal genomic sequence aligner for CPUs, GPUs, and ASICs.** Joël Lindegger, Damla Senol Cali, Mohammed Alser, Juan Gómez-Luna, Nika Mansouri Ghiasi and Onur Mutlu.

##### 2022

  - **Flexible seed size enables ultra-fast and accurate read alignment.** Kristoffer Sahlin.
  - **Sparse and skew hashing of k-mers.** Giulio Ermanno Pibiri.
  - **Metagenome assembly of high-fidelity long reads with hifiasm-meta.** Xiaowen Feng, Haoyu Cheng, Daniel Portik and Heng Li.
  - **Scalable sequence database search using Partitioned Aggregated Bloom Comb-Trees.** Camille Marchet and Antoine Limasset.
  - **Improved structural variant discovery in hard-to-call regions using sample-specific string detection from accurate long reads.** Luca Denti, Parsoa Khorsand, Paola Bonizzoni, Fereydoun Hormozdiari and Rayan Chikhi.
  - **The minimizer Jaccard estimator is biased and inconsistent.** Mahdi Belbasi, Antonio Blanca, Robert S. Harris, David Koslicki and Paul Medvedev.

##### 2021

  - **Accurate spliced alignment of long RNA sequencing reads.** Kristoffer Sahlin and Veli Mäkinen.
  - **Algorithms and complexity on indexing founder graphs.** Massimo Equi, Tuukka Norri, Jarno Alanko, Bastien Cazaux, Alexandru I. Tomescu and Veli Mäkinen.
  - **HolistIC: leveraging Hi-C and whole genome shotgun sequencing for double minute chromosome discovery.** Matthew Hayes, Angela Nguyen, Rahib Islam, Caryn Butler, Ethan Tran, Derrick Mullins and Chindo Hicks.
  - **CONSULT: Accurate contamination removal usinglocality-sensitive hashing.** Eleonora Rachtman, Vineet Bafna and Siavash Mirarab.
  - **DeepDRIM: a deep neural network to reconstruct cell-type-specific gene regulatory network using single-cell RNA-seq data.** Jiaxing Chen, Chinwang Cheong, Liang Lan, Xin Zhou, Jiming Liu, Aiping Lyu, William K Cheung and Lu Zhang.
  - **Conditional resampling improves calibration and sensitivity in single-cell CRISPR screen analysis.** Timothy Barry, Xuran Wang, Kathryn Roeder and Eugene Katsevich.
  - **Accel-Align: a fast sequence mapper and aligner based on the seed–embed–extend method.** Yiqing Yan, Nimisha Chaturvedi and Raja Appuswamy.
  - **BiTSC2: Bayesian inference of tumor clonal tree by joint analysis of single-cell SNV and CNA data.** Ziwei Chen, Fuzhou Gong, Lin Wan and Liang Ma.
  - **Automated strain separation in low-complexity metagenomes using long reads.** Riccardo Vicedomini, Christopher Quince, Aaron Darling and Rayan Chikhi.
  - **Real-time, direct classification of nanopore signals with SquiggleNet.** Yuwei Bao, Jack Wadden, John Erb-Downward, Piyush Ranjan, Robert Dickson, David Blaauw and Joshua Welch.
  - **Population-scale detection of non-reference sequence variants using colored de Bruijn graphs.** Thomas Krannich, W. Timothy J. White, Sebastian Niehus, Guillaume Holley, Bjarni V. Halldorsson and Birte Kehr.
  - **Ariadne: barcoded linked-read deconvolution using de Bruijn graphs.** Lauren Mak, Dmitry Meleshko, David C. Danko, Waris Barakzai, Natan Belchikov and Iman Hajirasouliha.
  - **ACTIVA: realistic single-cell RNA-seq generation with automatic cell-type identification using introspective variational autoencoders.** A. Ali Heydari, Oscar Davalos, Lihong Zhao, Katrina Hoyer and Suzanne Sindi.


#### Short Talks Selected from Abstracts

##### 2011

  - **A whole-genome probe design for massively parallel variant validation using selective circularization.** Daniel Newburger, Georges Natsoulis, Hua Xu, Sue Grimes, John Bell and Hanlee Ji.
  - **Accurate estimation of gene expression levels from DGE sequencing data.** Marius Nicolae and Ion Mandoiu.
  - **Constrained traversal of repeats with paired sequences.** Sébastien Boisvert, Élénie Godzaridis, François Laviolette and Jacques Corbeil.
  - **Contig graph mining for duplication breakpoints.** Jurgen F. Nijkamp, Jean-Marc Daran, Marcel J.T. Reinders and Dick De Ridder.
  - **Counting k-mers with a Bloom Filter.** Pall Melsted and Jonathan Pritchard.
  - **Finding deletions with exact break points from noisy low coverage paired-end short sequence reads.** Jin Zhang and Yufeng Wu.
  - **Improved variant discovery and allele frequency estimation from pooled dna resequencing with Bayesian latent class analysis and compositional bias models.** Shom Paul and Aaron Mackey.
  - **Modeling and automation of sequencing-based determination of RNA structure.** Sharon Aviran, Cole Trapnell, Julius Lucks, Stefanie Mortimer, Shujun Luo, Gary Schroth, Jennifer Doudna, Adam Arkin and Lior Pachter.
  - **mTiM: margin-based transcript mapping from RNA-seq.** Georg Zeller, Nico Goernitz, Gunnar Raetsch, Jonas Behr, Andre Kahles, Soeren Sonnenburg and Pramod Mudrakarta.
  - **Separating metagenomic data into genomes via clustering.** Olga Tanaseichuk and Tao Jiang.
  - **TavernaPBS: custom next-generation sequence analysis workflows using high-performance computing resources with Taverna and PBS.** Mark Lawson, Paul Shuber and Aaron Mackey.


#### Posters

##### 2025

 - **Accelerating gkm-SVM training through GPU implementation.** Dongwon Lee.
  - **Identifying potential therapeutic targets for heart failure through systematic transcriptome analysis.** Min-Ju Kim and Haeseung Lee.
  - **Population-specific and universal molecular features of skeletal muscle aging: Comparative transcriptome analysis of Korean and GTEx datasets.** Byeong-Don Min and Sang-Min Park.
  - **Unlocking hidden protein functions with a biochemically informed annotation strategy.** Olga Botvinnik.
  - **Unraveling miRNA-seq data: a statistical framework to account for competition for expression towards accurate differential expression analysis.** Seong-Hwan Jun.
  - **Systematic evaluation of dimensionality reduction methods for capturing transcriptomic signatures responding to drug treatments.** Yuseong Kwon, Sojeong Park, Soyoung Park and Haeseung Lee.
  - **GreedyMini: generating low-density DNA minimizers.** Shay Golan, Ido Tziony, Matan Kraus, Yaron Orenstein and Arseny Shur.
  - **GPU-accelerated homology search with MMseqs2.** Felix Kallenborn, Alejandro Chacon, Christian Hundt, Hassan Sirelkhatim, Kieran Didi, Sooyoung Cha, Christian Dallago, Milot Mirdita, Bertil Schmidt and Martin Steinegger .
  - **strangepg: toward pangenome scale graph visualization.** Konstantinn Bonne and Tobias Marschall.
  - **Vizitig: context-rich exploration of sequencing datasets.** Bastien Degardins, Charles Paperman and Camille Marchet.
  - **Identifying gene-environment interactions for cancer incidence using epigenomic profiles.** Younghoon Kim.
  - **stDyer enables spatial domain clustering with dynamic graph embedding.** Ke Xu, Yu Xu, Zirui Wang, Xin Zhou and Lu Zhang.
  - **Splicing junction classifier for detecting abnormal KEAP1-NRF2 system activation.** Raul Mateos, Wira Winardi, Kenichi Chiba, Ai Okada, Ayako Suzuki, Yoichiro Mitsuishi and Yuichi Shiraishi.
  - **Adapting broad protein language models to viruses.** Spyros Lytras, Adam Strange, Jumpei Ito and Kei Sato.
  - **b-move: faster lossless approximate pattern matching in a run-length compressed index.** Lore Depuydt, Luca Renders, Simon Van de Vyver, Lennart Veys, Travis Gagie and Jan Fostier.
  - **Full length isoform reconstruction in single cell data.** Marie Van Hecke, Koen Deserranno, Elise Callens, Filip Van Nieuwerburgh and Kathleen Marchal.
  - **A novel computational pipeline for the functional characterization and deorphanization of G-protein coupled receptors.** Catherine Zhou.
  - **Parallel and space efficient exact local alignment.** Evelin Aasna.
  - **Pre-training dataset deduplication improves genomic LLMs.** Mahler Revsine, Daniel Khashabi and Michael Schatz.

##### 2023

  - **SeGraM: a universal hardware accelerator for genomic sequence-to-graph and sequence-to-sequence mapping.** Damla Senol Cali, Konstantinos Kanellopoulos, Joel Lindegger, Zülal Bingöl, Gurpreet Singh Kalsi, Ziyi Zuo, Can Firtina, Meryem Banu Cavlak, Jeremie S. Kim, Nika Mansouri Ghiasi, Gagandeep Singh, Juan Gómez Luna, Nour Almadhoun Alserr, Mohammed Alser, Sreenivas Subramoney, Can Alkan, Saugata Ghose and Onur Mutlu.
  - **Detection of large tandem duplications in HMPV isolates.** Thomas Krannich, Stephan Fuchs and Sophie Köndgen.
  - **Predicting the origin of soil samples - performance evaluation of a new targeted high-throughput sequencing metagenomic tool..** Kamila Marszałek, Michał B. Kowalski, Andrzej Ossowski, Rafał Płoski, Renata Zbieć Piekarska, Paweł P. Łabaj and Wojciech Branicki.
  - **A rigorous benchmarking of methods for SARS-CoV-2 lineage abundance estimation in wastewater.** Viorel Munteanu, Khooshbu Kantibhai Patel, Nitesh Kumar Sharma, Sergey Knyazev and Serghei Mangul.
  - **Rigorous benchmarking of HLA callers for RNA-seq data.** Ram Ayyala, Dottie Yu, Sergey Knyazev and Serghei Mangul.
  - **RNA-Seq-based methods are able to effectively capture the clonotypes and estimate the diversity of TCR repertoires in T cell rich tissues and certain repertoires.** Serghei Mangul.
  - **Machine learning enabled pattern discovery in large-scale spatial gene expression datasets.** Reza Abbasi-Asl.
  - **Assessing the completeness of immunogenetics databases across diverse populations.** Yu-Ning Huang, Yiting Meng, Naresh Amrat Patel, Jay Himanshu Mehta, Brittney Hua, Marina Fayzullina, Houda Alachkar and Serghei Mangul.
  - **The systematic assessment of completeness of public metadata accompanying omics studies.** Yu-Ning Huang, Anushka Rajesh, Ram Ayyala, Aditya Sarkar, Ruiwei Guo, Irina Nakashidze, Shirley Monge, Dottie Yu, Qiushi Peng, Grace Scheg, Khooshbu Kantibhai Patel, Tejasvene Ramesh, Anushka Yadav, Fangyun Liu, Jay Himanshu Mehta and Serghei Mangul.
  - **SurfR: Surfing the cells' surfaceome.** Aurora Maurizio, Anna Sofia Tascini and Marco Jacopo Morelli.
  - **GoPeaks: histone modification peak calling for CUT&Tag.** William Yashar, Garth Kong, Jake Vancampen, Brittany Curtiss, Daniel Coleman, Lucia Carbone, Galip Yardimci, Julia Maxson and Theodore Braun.
  - **ClusterV: accurate detection of HIV quasispecies and drug resistance mutations using ONT sequencing data.** Junhao Su, Tak-Wah Lam and Ruibang Luo.
  - **Unikseq: unique region identification in genome sequences using a k-mer approach, to empower environmental DNA assay designs and comparative genomics studies.** Rene Warren, Michael J Allison, M. Louie Lopez, Neha Acharya-Patel, Lauren Coombe, Cecilia L. Yang, Caren C Helbing and Inanc Birol.
  - **ntHits: streaming through raw sequencing data to profile and filter k-mers with selected multiplicities.** Parham Kazemi, Hamid Mohamadi, Justin Chu, Lauren Coombe, Rene L Warren and Inanc Birol.
  - **Evaluating the Robustness and Reproducibility of RNA-Seq Quantification Tools.** Fangyun Liu, Brian Nadel, Pelin Icer Baykal and Serghei Mangul.
  - **Copy number estimation using Counting Bloom Filters in de novo assembled genomes.** Klea Zambaku, Ricardo Roman-Brenes, Ömer Yavuz Öztürk, Can Alkan and Inanç Birol.
  - **Improving functional annotation of bacterial genomes with COGtools.** Karel Sedlar, Petra Polakovicova and Ralf Zimmer.
  - **Orthanq: orthogonal evidence based haplotype quantification.** Hamdiye Uzuner and Johannes Köster.
  - **Minichain: a new method for pangenome graph construction.** Ghanshyam Chandra and Chirag Jain.
  - **ALIBI2: improved linearization of pangenome graphs.** Anna Lisiecka and Norbert Dojer.
  - **Characterization of alignment and search algorithms for short read, long read, and graph mappers.** Ecem İlgün, Ömer Yavuz Öztürk, Klea Zambaku, Juan Gómez Luna, Mohammed Alser, Ricardo Roman-Brenes, Can Alkan and The Biopim Project.
  - **Nanopore signal alignment, analysis, and visualization with Uncalled4.** Sam Kovaka, Paul W. Hook, Vikram Shivakumar, Katharine M. Jenike, Luke Morina, Roham Razaghi, Winston Timp and Michael C. Schatz.
  - **Mod.Plot: a rapid and interactive visualization of tandem repeats.** Alexander Sweeten, Adam Phillippy and Michael Schatz.
  - **Using minimizer interarrival distances for read-until human read detection from blood samples sequenced by Oxford Nanopore.** Sina Barazandeh, Mahmud Sami Aydin, Berke Ucar, Can Alkan and Inanc Birol.

##### 2022

  - **Finding Significant Genes and Pathways Across Viruses in CRISPR-Cas9 Screen Database.** Elianna Kondylis, Jacklyn Luu, Kyle Awayan, Andreas Puschnik, Angela Pisco.
  - **Metabuli: a metagenomic classifier that combines protein- and DNA-level classification to achieve both high sensitivity and specificity.** Jaebeom Kim and Martin Steinegger.
  - **Sketching and sampling approaches for fast and accurate long read classification.** Arun Das and Michael Schatz.
  - **Theory of local k-mer selection with applications to long-read alignment.** Jim Shaw and Yun William Yu.
  - **Metagenome assembly of high-fidelity long reads with hifiasm-meta.** Xiaowen Feng, Haoyu Cheng, Daniel Portik and Heng Li.
  - **Automated telomere-to-telomere genome assembly with PacBio HiFi and ultra-long ONT data.** Mikko Rautiainen, Sergey Nurk, Brian Walenz, Adam M Phillippy, Sergey Koren.
  - **Unlocking the microblogging potential for science and medicine.** Karishma Chhugani and Serghei Mangul.

##### 2018

  - **mirLibSpark: a scalable NGS microRNA prediction pipeline with data aggregation.** Chao-Jung Wu, Mohamed Amine Remita and Abdoulaye Baniré Diallo.
  - **K-merator, an efficient design of highly specific k-mers for quantification of transcriptional signatures in large scale RNAseq cohorts..** Sébastien Riquier, Anne-Laure Bougé, Benoit Guibert, Jérôme Audoux, Daniel Gautheret, Thérèse Commes and Anthony Boureux.
  - **Kevlar: Mapping-free approach for accurate discovery of de novo variants.** Daniel Standage, C. Titus Brown and Fereydoun Hormozdiari.
  - **Promoter and enhancer chromatin dynamics during pancreatic differentiation.** Henriette Miko, Scott A. Lacadie and Uwe Ohler.
  - **Ultrafast space-efficient k-mer indexing.** Sven Rahmann.
  - **ARKS: chromosome-scale human genome scaffolding with linked read kmers.** Rene Warren, Lauren Coombe, Jessica Zhang, Ben Vandervalk, Justin Chu, Shaun Jackman and Inanc Birol.
  - **Tigmint: correct assembly errors using linked reads from large molecules.** Shaun Jackman, Lauren Coombe, Justin Chu, Rene Warren, Ben Vandervalk, Sarah Yeo, Zhuyi Xue, Hamid Mohamadi, Joerg Bohlmann, Steven Jones and Inanc Birol.
  - **Multi-Index Bloom Filters: A probabilistic data structure for sensitive multi-reference sequence classification with multiple spaced seeds.** Justin Chu, Emre Erhan, Hamid Mohamadi, Ben Vandervalk, Jeffrey Tse, Sarah Yeo, Shaun Jackman, Ka Ming Nip, Rene Warren and Inanc Birol.
  - **De novo clustering of gene expressed variants in transcriptomic long reads data sets.** Camille Marchet, Lolita Lecompte, Jean-Marc Aury, Corinne Da Silva, Corinne Cruaud, Jacques Nicolas and Pierre Peterlongo.
  - **ONTig: contiguating genome assembly using oxford nanopore long reads.** Hamid Mohamadi, Ben Vandervalk, Shaun Jackman, Lauren Coombe, Justin Chu, Rene Warren and Inanc Birol.
  - **Rapid and precise analysis of human gut metagenomes using Oxford Nanopore sequencing technology.** Hugo Roume, Mathieu Almeida, Florian Plaza Oñate and S. Dusko Ehrlich.
  - **S3A: a scalable and accurate annotated assembly tool for targeted gene assembly.** Laurent David, Hugues Richard, Riccardo Vicedomini and Alessandra Carbone.
  - **Pan-genome structural analysis and visualisation.** Paulina Dziadkiewicz, Jakub Tyrek and Norbert Dojer.
  - **Reference-guided genome assembly in metagenomic samples.** Cervin Guyomar, Wesley Delage, Fabrice Legeai, Christophe Mougel, Jean-Christophe Simon and Claire Lemaitre.
  - **Nonparametric identification of epigenomic networks from large-scale ChIP-seq experiments.** Gabriele Schweikert and Sanguinetti Guido.
  - **Accelerating approximate pattern matching with processing-in-memory (PIM) and single-instruction multiple-data (SIMD) programming.** Damla Senol Cali, Zulal Bingol, Jeremie Kim, Rachata Ausavarungnirun, Saugata Ghose, Can Alkan and Onur Mutlu.
  - **Isoform assembly with quasi-lossless compression of quality scores in RNA-seq data.** Ana Hernandez.
  - **Map2Peak: from unmapped reads to CHIP-seq peaks in half the time.** Krishna Reddy Gujjula and Kiavash Kianfar.

##### 2015

  - **Deep sequencing characterization of Sus scrofa piRNA fraction shared between female and male gonads.** Aleksandra Swiercz, Dorota Kowalczywiewicz, Luiza Handschuh, Katarzyna Lesniak, Marek Figlerowicz and Jan Wrzesinski.
  - **GPU-accelerated whole genome assembly.** Michał Kierzynka, Wojciech Frohmberg, Jacek Błażewicz, Piotr Żurkowski, Marta Kasprzak and Paweł Wojciechowski.
  - **Scaling ABySS to longer reads using spaced k-mers and Bloom filters.** Shaun Jackman, Karthika Raghavan, Benjamin Vandervalk, Daniel Paulino, Justin Chu, Hamid Mohamadi, Anthony Raymond, Rene Warren and Inanc Birol.
  - **Conditional entropy in variation-adjusted windows detects positive selection signatures relevant to next Generation sequencing.** Samuel K. Handelman, Michal Seweryn, Ryan M. Smith, Katherine Hartmann, Danxin Wang, Maciej Pietrzak, Andrew D. Johnson, Andrzej Kloczkowski, Wolfgang Sadee.

##### 2012

  - **Detection of chromosomal inversions with paired-end sequencing.** José Ignacio Lucas Lledó and Mario Cáceres.
  - **A context-based approach to identify the most likely mapping for RNA-seq experiments.** Thomas Bonfert, Gergely Csaba, Ralf Zimmer and Caroline C. Friedel.
  - **Understanding the nucleation of the microRNA-mRNA pairing by using CLIP-Seq and RNA folding data.** Ray Marin and Jiri Vanicek.
  - **Calling inversions from next-generation sequencing paired-end mapping data with GRIAL.** Sònia Casillas, Can Alkan, Evan E Eichler and Mario Cáceres.
  - **Empirical evaluation of different modern reference panels for imputation and their implication for Genome Wide Association Studies..** Sílvia Bonàs, Josep M. Mercader and David Torrents.
  - **Filtering duplicate reads from 454 pyrosequencing data.** Susanne Balzer, Ketil Malde, Inge Jonassen and Markus A. Grohme.
  - **Improving loss of heterozygosity identification by tumor purity estimation.** Eva König, Lars Feuerbach, Barbara Hutter, Matthias Schlesner, Qi Wang, Benedikt Brors and Thomas Lengauer.
  - **A probabilistic method for structural variant prediction from strobe sequencing data.** Anna Ritz, Suzanne Sindi, Ali Bashir and Benjamin Raphael.
  - **Integrative de novo transcriptome assembly in fruit fly.** Nathan Boley.
  - **The devil is in the detail: mining and annotating genomic variants in the Tasmanian Devil facial tumour genome.** Ole Schulz-Trieglaff, Elizabeth Murchison, Zemin Ning and Anthony Cox.
  - **Automated workflow for RNA-Seq analysis: application and testing with various types of RNA-Seq protocols.** Irina Khrebtukova, Ryan Kelley, Shujun Luo, Tim Hill, Patrick Lau, Jennifer Chiniquy, Kathryn Stephens, Semyon Kruglyak and Gary P Schroth.
  - **The de novo Genome Assembly Assessment Server.** André Corvelo and Tyler Alioto.
  - **Improving RNA sequencing interpretation: a case study on breast cancer cell lines.** Kirstine Belling, David Flores, Daniel Elias, Jan Stenvang, Jun Wang, Nils Brünner, Henrik Ditzel and Ramneek Gupta.
  - **Coalescing discordant read mapping signatures for structural variant breakpoint detection.** Ryan M. Layer, Aaron R. Quinlan and Ira M. Hall.
  - **Analysis of pandemic (H1N1) 2009 Influenza A virus circulating in Mexico during the 2011-2012 season by ultra-deep sequencing.** Joanna Ortiz Alcantara, Elizabeth González Durán, Araceli Rodriguez Castillo, Fabiola Garcés Ayala, José Miguel Segura Candelas, Claudia Wong Arámbula, Patricia Alcántara Pérez, Abril Rodríguez, Brisia Rodríguez, Juan Carlos Del Mazo, Susana Serrano, Gisela Barrera Badillo, Irma López Martínez, Lucía Hernández Rivas, Hugo López-Gatell, Celia Alpuche Aranda and José Ernesto Ramírez González.
  - **YAHA: fast and flexible long-read alignment with optimal breakpoint detection.** Gregory Faust and Ira Hall.
  - **Efficient and error-tolerant sequencing read mapping.** Norbert Dojer and Piotr Jaroszyński.
  - **Identifying genomic copy number alteration and loss of heterozygosity in next-generation sequence data.** John R. McPherson, Yingting Wu, Patrick Tan and Steve Rozen.
  - **Algorithms to find mutated pathways in cancer.** Fabio Vandin, Hsin-Ta Wu, Eli Upfal and Ben Raphael.
  - **Torrent Variant Caller: it’s all about speed, accuracy, and long indels.** Dumitru Brinza, Zheng Zhang, Eric Tsung, Charles Scafe, Onur Sakarya, Alexander Joyner, Sowmi Utiramerur, Guy Del Mistro, Fiona Hyland and Ellen Beasley.
  - **The GEM toolkit: world-class short read mapping, and more.** Santiago Marco Sola and Paolo Ribeca.
  - **Strategies for sequencing and analysis of low-diversity samples.** Maga Rowicka and Abhishek Mitra.
  - **Instant-Seq:- an integrated tool with web interface for fast analysis of ChIP-Seq data.** Abhishek Mitra and Maga Rowicka.
  - **Quasispecies spectrum reconstruction using multi-commodity flows.** Nicholas Mancuso, Bassam Tork, Pavel Skums, Ion Mandoiu and Alex Zelikovsky.
  - **On the comparison of sets of alternative transcripts.** Aida Ouangraoua, Krister Swenson and Anne Bergeron.
  - **CNVeM: Copy number variation detection using uncertainty of read mapping.** Zhanyong Wang, Farhad Hormozdiari, Wen-Yun Yang, Eran Halperin and Eleazar Eskin.
  - **GRAPE RNAseq analysis pipeline environment.** David Gonzalez-Knowles, Maik Roder, Angelika Merkel and Roderic Guigo.

##### 2011

  - **Epigenetics of atherosclerosis.** Lauren Mills, Brian Wamhoff, Brett Blackman, Aaron Mackey and Jessica Connelly.
  - **Genome-scale analysis of promoter melting in eukaryotic gene transcription.** Fedor Kouzine, Damian Wójtowicz, Arito Yamane, Wolfgang Resch, Teresa M. Przytycka, David Levens and Rafael Casellas.
  - **Haplotype discovery based on unassembled sequences estimation.** Serghei Mangul and Alex Zelikovsky.
  - **RGASP evaluation of RNA-seq read alignment algorithms.** Andre Kahles, Regina Bohnert, Paolo Ribeca, Jonas Behr and Gunnar Raetsch.
  - **Screening for transposable element-induced adaptations in Drosophila melanogaster using next-gen sequencing data.** Anna-Sophie Fiston-Lavier, Dmitri Petrov and Josefa Gonzalez.
  - **SeqMDD: symbolic data structures for accurate mapping.** Marco Beccuti, Francesca Cordero, Susanna Donatelli and Raffaele Calogero.
  - **SlideSort: A fast and exact tool for finding all similar pairs from next-generation sequencing data.** Kana Shimizu and Koji Tsuda.
  - **Smarti - A fast short read alignment algorithm.** Florian Schatz, Sascha Möller and Manfred Schimmler.
  - **Toward assessing the quality of *de novo* assembly.** Rasiah Loganantharaj.

