<!-- Begin Schedule Section -->
<section id="schedule" class="schedule">
    <div class="content-wrapper text-left">
        <div class="col-md-8 col-sm-8 col-lg-8 col-lg-offset-2 col-md-offset-2 col-sm-offset-1">

            <h2>Program of RECOMB-seq 2025</h2>

            <h2>April 24th, Thursday</h2>
            <table border="1" style="width: 100%; table-layout: fixed; border-collapse: collapse;">
                <thead>
                    <tr style="text-align: left;">
                        <th style="width:17%; padding:10px;">Time/Day</th>
                        <th style="width:58%; padding:10px;">Event</th>
                        <th style="width:25%; padding:10px;">Speaker/Details</th>
                    </tr>
                </thead>
                <tbody>
          
    <tr>
      <td>8:30-9:30 am</td>
      <td><b>Registration</b></td>
      <td>The Commons, Yonsei University</td>
    </tr>
    <tr>
      <td>9:25-9:30 am</td>
      <td><b>Opening</b></td>
      <td>B125 Grand Ballroom</td>
    </tr>
    <tr>
      <td>9:30-10:30 am</td>
      <td><b>Keynote 1: From Petabytes to Insights: Scaling Sequence Bioinformatics with Logan</b></td>
      <td>Rayan Chikhi, Institut Pasteur</td>
    </tr>
    <tr>
      <td>10:30-10:45 am</td>
      <td><b>Coffee break</b></td>
      <td></td>
    </tr>
    <tr>
      <td><b>Session 1: Long talks - Proceedings </b></td>
      <td></td>
      <td>Session chair: Rayan Chikhi</td>
    </tr>
    <tr>
      <td>10:45-11:10 am</td>
      <td>Tile-X: A vertex reordering approach for scalable long read assembly [Proceedings]</td>
      <td>Oieswarya Bhowmik, Washington State University</td>
    </tr>
    <tr>
      <td>11:10-11:35 am</td>
      <td>Run-length compressed metagenomic read classification with SMEM-finding and tagging [Proceedings]</td>
      <td>Lore Depuydt, Ghent University</td>
    </tr>
    <tr>
      <td>11:35-12:00 am</td>
      <td>Alignment-Free Detection of Differences Between Sequencing Data Sets [Proceedings]</td>
      <td>Alessia Petescia	Comenius, University in Bratislava</td>
    </tr>
    <tr>
      <td>12:00-1:30 pm</td>
      <td><b>Lunch / Poster viewing (odd numbers)</b></td>
      <td></td>
    </tr>
    <tr>
      <td><b>Session 2: Short talks<b/></td>
      <td></td>
      <td>Session chair: Milot Mirdita</td>
    </tr>
    <tr>
      <td>1:30-1:42 pm</td>
      <td>Metrics Matter: Why We Need to Stop Using Silhouette in Single-Cell Benchmarking</td>
      <td>Pia Rautenstrauch, MDC Berlin</td>
    </tr>
    <tr>
      <td>1:42-1:54 pm</td>
      <td>      Full length isoform reconstruction in single cell data</td>
      <td>Marie Van Hecke, Ghent University - imec</td>
    </tr>
    <tr>
      <td>1:55-2:07 pm</td>
      <td>      Masked superstrings as a compact, indexable, and dynamic representation of unconstrained k-mer sets</td>
      <td>Pavel Veselý, Charles University</td>
    </tr>
    <tr>
      <td>2:07-2:19 pm</td>
      <td>      Automated Annotation of Satellite DNA</td>
      <td>Alex Sweeten, Johns Hopkins University</td>
    </tr>
    <tr>
      <td>2:20-2:40 pm</td>
      <td><b>Break</b></td>
      <td></td>
    </tr>
    <tr>
      <td><b>Session 3: Short talks</b></td>
      <td></td>
      <td>Session chair: Chirag Jain</td>
    </tr>
    <tr>
      <td>2:40-2:52 pm</td>
      <td>      b-move: Faster Lossless Approximate Pattern Matching in a Run-Length Compressed Index	</td>
      <td>Lore Depuydt, Ghent University</td>
    </tr>
    <tr>
      <td>2:52-3:04 pm</td>
      <td>      strangepg: Toward Pangenome Scale Graph Visualization	</td>
      <td>Konstantinn Bonnet, Heinrich Heine University</td>
    </tr>
    <tr>
      <td>3:05-3:17 pm</td>
      <td>      Improved variant calling via latent breakpoint graphs	</td>
      <td></td>
    </tr>
    <tr>
      <td>3:17-3:29 pm</td>
      <td>      A Novel K-mer Masking Approach for Improving Specificity in Metagenomic Pathogen Detection	</td>
      <td>Joanne Qiu, UCLA</td>
    </tr>
    <tr>
      <td>3:30-3:45 pm</td>
      <td><b>Break</td>
      <td></td>
    </tr>
    <tr>
      <td><b>Session 4: Long talks - Overlay</b></td>
      <td></td>
      <td>Session chair: Pavel Veselý</td>
    </tr>
    <tr>
      <td>3:45-4:10 pm</td>
      <td>DeepMM: Identify and correct Metagenome Misassemblies with deep learning [Overlay]</td>
      <td>Ding Yi, Hong Kong Baptist University</td>
    </tr>
    <tr>
      <td>4:10-4:35 pm</td>
      <td>GPU-accelerated homology search with MMseqs2 [Overlay]</td>
      <td>Milot Mirdita, Seoul National University</td>
    </tr>
    <tr>
      <td>4:35-5:00 pm</td>
      <td>On the Coverage Required for Diploid Genome Assembly [Overlay]</td>
      <td>Chirag Jain, Indian Institute of Science</td>
    </tr>
       <tr>
      <td>5:00-5:15 pm</td>
      <td><b>Poster highlights</b></td>
      <td></td>
    </tr>
    <tr>
      <td>5:15 pm</td>
      <td><b>Poster viewing (odd numbers)</b></td>
      <td>Slots 101 to 120</td>
    </tr>
   
  </tbody>
</table>

<br><br>

<h2>April 25th, Friday</h2>
            <table border="1" style="width: 100%; table-layout: fixed; border-collapse: collapse;">
                <thead>
                    <tr style="text-align: left;">
                        <th style="width:17%; padding:10px;">Time/Day</th>
                        <th style="width:58%; padding:10px;">Event</th>
                        <th style="width:25%; padding:10px;">Speaker/Details</th>
                    </tr>
                </thead>
                <tbody>
    <tr>
      <td>8:30-9:30 am</td>
      <td><b>Registration</b></td>
      <td>The Commons, Yonsei University</td>
    </tr>
    <tr>
      <td>9:25-9:30 am</td>
      <td><b>Opening</td></td>
      <td>B125 Grand Ballroom</td>
    </tr>
    <tr>
      <td>9:30-10:30 am</td>
      <td><b>Keynote 2: Single cells and long reads: Transcriptomes at exquisite resolution</b></td>
      <td>Alicia Oshlack, Peter MacCallum Cancer Center</td>
    </tr>
    <tr>
      <td>10:30-10:45 am</td>
      <td><b>Coffee break</b></td>
      <td></td>
    </tr>
    <tr>
      <td><b>Session 5: Long talks - Proceedings</b></td>
      <td></td>
      <td>Session chair: Alicia Oshlack</td>
    </tr>
    <tr>
      <td>10:45-11:10 am</td>
      <td>Decoding the causal drivers of spatial cellular topology  [Proceedings]</td>
      <td>Rohit Singh, Duke</td>
    </tr>
    <tr>
      <td>11:10-11:35 am</td>
      <td>Efficient trace reconstruction in DNA storage systems using Bidirectional Beam Search [Proceedings]</td>
      <td>Gu Zhenhao, National University of Singapore</td>
    </tr>
    <tr>
      <td>11:35-12:00 am</td>
      <td>Decoupling Hi-C matrices with non-negative matrix factorization to reveal assorted TADs [Proceedings]</td>
      <td>Zhao Ling, City university of HongKong</td>
    </tr>
    <tr>
      <td>12:00-1:30 pm</td>
      <td><b>Lunch / Poster viewing (even numbers)</b></td>
      <td></td>
    </tr>
    <tr>
      <td><b>Session 6: Short talks</b></td>
      <td></td>
      <td>Session chair: Rohit Singh</td>
    </tr>
    <tr>
      <td>1:30-1:42 pm</td>
      <td>      Pre-Training Dataset Deduplication Improves Genomic LLMs</td>
      <td>Mahler Revsine, Johns Hopkins University</td>
    </tr>
    <tr>
      <td>1:42-1:54 pm</td>
      <td>      Edgecopy: Accurate CNV calling in duplicated genes using whole-exome sequencing</td>
      <td>Sang Yoon Byun, University of California San Diego</td>
    </tr>
    <tr>
      <td>1:55-2:07 pm</td>
      <td>  Reindeer2: practical abundance index at scale			</td>
      <td> Camille Marchet, Université de Lille - CNRS</td>
    </tr>
    <tr>
      <td>2:07-2:19 pm</td>
      <td>      Inverted colored de Bruin Graph for practical kmer set storage</td>
      <td>Timothé Rouzé, Institut Pasteur </td>
    </tr>
    <tr>
      <td>2:20-2:40 pm</td>
      <td><b>Coffee break</b></td>
      <td></td>
    </tr>
    <tr>
      <td><b>Session 7: Short talks</b></td>
      <td></td>
      <td>Session chair: Luca Denti</td>
    </tr>
    <tr>
      <td>2:40-2:52 pm</td>
      <td>   Vizitig: context-rich exploration of sequencing datasets 	</td>
      <td>Camille Marchet, Université de Lille - CNRS</td>
    </tr>
    <tr>
      <td>2:52-3:04 pm</td>
      <td>      Multi-sample, multi-platform isoform quantification using empirical Bayes			</td>
      <td>Arghamitra Talukder, Columbia University</td>
    </tr>
    <tr>
      <td>3:05-3:17 pm</td>
      <td>      Efficient algorithm for resolving scenarios of Complex Chromosomal Rearrangements			</td>
      <td>Barbara Poszewiecka, University of Warsaw</td>
    </tr>
    <tr>
      <td>3:17-3:29 pm</td>
      <td>      De Bruijn Graphs for Pangenomics: In-depth Performance Benchmarking of de Bruijn Graph-Based Tools for Read Mapping		</td>
      <td>Zülal Bingöl, Bilkent University</td>
    </tr>
    <tr>
      <td>3:30-3:45 pm</td>
      <td><b>Coffee break</b></td>
      <td></td>
    </tr>
    <tr>
      <td><b>Session 8: Long talks - Overlay</b></td>
      <td></td>
      <td>Session chair: Yaron Orenstein</td>
    </tr>
    <tr>
      <td>3:45-4:10 pm</td>
      <td>Pangenome graph augmentation from unassembled long reads [Overlay]</td>
      <td>Luca Denti, Comenius University in Bratislava</td>
    </tr>
    <tr>
      <td>4:10-4:35 pm</td>
      <td>De novo clustering of extensive long-read transcriptome datasets with isONclust3 [Overlay]</td>
      <td>Alexander Petri, Stockholm University</td>
    </tr>
    <tr>
      <td>4:35-5:00 pm</td>
      <td>Multi-context seeds enable fast and high-accuracy read mapping [Overlay]</td>
      <td></td>
    </tr>
    <tr>
      <td>17:00</td>
      <td><b>Poster viewing (even numbers)</b></td>
      <td> Slots 101 to 120</td>
    </tr>
  </tbody>
</table>



<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>





		<h4>Accepted Papers to the Proceedings Track</h4>

		<ul style="list-style-type: disc;">
		<li>Alessia Petescia, Luca Denti, Askar Gafurov, Viktória Hodorová, Jozef Nosek, Broňa Brejová and Tomas Vinar	Alignment-Free Detection of Differences Between Sequencing Data Sets</li>
		<li>Oieswarya Bhowmik and Ananth Kalyanaraman	Tile-X: A vertex reordering approach for scalable long read assembly</li>
		<li>Prannav Shankar, Huan Liang, Uthsav Chitra and Rohit Singh 	Decoding the causal drivers of spatial cellular topology </li>
		<li>Zhenhao Gu, Hongyi Xin, Puru Sharma, Gary Yipeng Goh, Limsoon Wong and Niranjan Nagarajan	Efficient trace reconstruction in DNA storage systems using Bidirectional Beam Search</li>
		<li>Zhao Ling, Shi Ying Li, Jingwan Wang, Xikang Feng, Bowen Tan and Shuai Cheng Li	Decoupling Hi-C matrices with non-negative matrix factorization to reveal assorted TADs </li>
		<li>Lore Depuydt, Omar Y. Ahmed, Jan Fostier, Ben Langmead and Travis Gagie	Run-length compressed metagenomic read classification with SMEM-finding and tagging</li>
		</ul>
		
		<h4>Accepted Papers to the Overlay Track</h4>
		<ul style="list-style-type: disc;">
		<li>Alexander Petri and Kristoffer Sahlin	De novo clustering of extensive long-read transcriptome datasets with isONclust3</li>
		<li>Ivan Tolstoganov, Marcel Martin and Kristoffer Sahlin	Multi-context seeds enable fast and high-accuracy read mapping</li>
		<li>Felix Kallenborn, Alejandro Chacon, Christian Hundt, Hassan Sirelkhatim, Kieran Didi, Sooyoung Cha, Christian Dallago, Milot Mirdita, Bertil Schmidt and Martin Steinegger	GPU-accelerated homology search with MMseqs2</li>
		<li>Yi Ding, Jin Xiao, Bohao Zou and Lu Zhang	DeepMM: Identify and correct Metagenome Misassemblies with deep learning</li>
		<li>Daanish Mahajan, Chirag Jain and Navin Kashyap	On the Coverage Required for Diploid Genome Assembly </li>
		<li>Luca Denti, Paola Bonizzoni, Brona Brejova, Rayan Chikhi, Thomas Krannich, Tomas Vinar and Fereydoun Hormozdiari	Pangenome graph augmentation from unassembled long reads </li>
		</ul>

		<h4>Accepted Short Talks</h4>
		<ul style="list-style-type: disc;">
		<li>Pia Rautenstrauch and Uwe Ohler	Metrics Matter: Why We Need to Stop Using Silhouette in Single-Cell Benchmarking</li>
		<li>Marie Van Hecke, Koen Deserranno, Elise Callens, Filip Van Nieuwerburgh and Kathleen Marchal	Full length isoform reconstruction in single cell data</li>
		<li>Ondřej Sladký, Pavel Veselý and Karel Brinda	Masked superstrings as a compact, indexable, and dynamic representation of unconstrained k-mer sets</li>
		<li>Alexander Sweeten, Adam Phillippy and Michael Schatz	Automated Annotation of Satellite DNA</li>
		<li>Lore Depuydt, Luca Renders, Simon Van de Vyver, Lennart Veys, Travis Gagie and Jan Fostier	b-move: Faster Lossless Approximate Pattern Matching in a Run-Length Compressed Index</li>
		<li>Konstantinn Bonnet and Tobias Marschall	strangepg: Toward Pangenome Scale Graph Visualization</li>
		<li>Megan Le, Lillian Zhang, Can Koçkan, Barış Ekim, Houlin Yu, Brian Haas, Aziz Al'Khafaji, Bonnie Berger and Victoria Popic	Improved variant calling via latent breakpoint graphs</li>
		<li>Junqiong Qiu, Seungmo Lee, Vivek Agarwal and William O'Brien	A Novel K-mer Masking Approach for Improving Specificity in Metagenomic Pathogen Detection</li>
		<li>Mahler Revsine, Daniel Khashabi and Michael Schatz	Pre-Training Dataset Deduplication Improves Genomic LLMs</li>
		<li>Sang Yoon Byun and Vikas Bansal	Edgecopy: Accurate CNV calling in duplicated genes using whole-exome sequencing</li>
		<li>Bastien Degardins, Charles Paperman and Camille Marchet	Vizitig: context-rich exploration of sequencing datasets</li>
		<li>Timothé Rouzé, Rayan Chikhi and Antoine Limasset	Inverted colored de Bruin Graph for practical kmer set storage</li>
		<li>Yohan Hernandez Courbevoie, Mikaël Salson, Chloé Bessière, Haoliang Xue, Daniel Gautheret, Camille Marchet and Antoine Limasset	Reindeer2: practical abundance index at scale</li>
		<li>Arghamitra Talukder, Shree Thavarekere, Madison Mehlferber, Gloria M Sheynkman and David A. Knowles	Multi-sample, multi-platform isoform quantification using empirical Bayes</li>
		<li>Zülal Bingöl, Berkan Şahin, Konstantina Koliogeorgi, Ricardo Roman-Brenes, Klea Zambaku, Can Firtina, Onur Mutlu and Can Alkan	De Bruijn Graphs for Pangenomics: In-depth Performance Benchmarking of de Bruijn Graph-Based Tools for Read Mapping</li>
		<li>Barbara Poszewiecka, Krzysztof Gogolewski and Anna Gambin	Efficient algorithm for resolving scenarios of Complex Chromosomal Rearrangements</li>
		</ul>


		<h4>Accepted Posters</h4>
		<ul style="list-style-type: disc;">
		<li>101. Accelerating gkm-SVM Training Through GPU Implementation</li>
		<li>102. Identifying Potential Therapeutic Targets for Heart Failure through Systematic Transcriptome Analysis, Hello, Kim Min-Ju, Department of Pharmacy, Pusan National University</li>
<li>103. Population-specific and universal molecular features of skeletal muscle aging: Comparative transcriptome analysis of Korean and GTEx datasets, Byeong-Don Min, Chungnam National University</li>
<li>104. Unlocking Hidden Protein Functions with a Biochemically Informed Annotation Strategy</li>
<li>105. Unraveling miRNA-seq data: a statistical framework to account for competition for expression towards accurate differential expression analysis</li>
<li>106. Systematic evaluation of dimensionality reduction methods for capturing transcriptomic signatures responding to drug treatments</li>
<li>107. GreedyMini: Generating low-density DNA minimizers, Yaron Orenstein, Bar-Ilan University</li>
<li>108. GPU-accelerated homology search with MMseqs2</li>
<li>109. strangepg: Toward Pangenome Scale Graph Visualization, Konstantinn Bonnet, Heinrich Heine University</li>
<li>110. Vizitig: context-rich exploration of sequencing datasets, Camille Marchet, Université de Lille - CNRS</li>
<li>111. Identifying gene-environment interactions for cancer incidence using epigenomic profiles, Younghoon Kim, KISTI</li>
<li>112. stDyer enables spatial domain clustering with dynamic graph embedding</li>
<li>113. Splicing Junction Classifier for Detecting Abnormal KEAP1-NRF2 System Activation</li>
<li>114. Adapting broad protein language models to viruses</li>
<li>115. b-move: Faster Lossless Approximate Pattern Matching in a Run-Length Compressed Index, Lore Depuydt, Ghent University</li>
<li>116. Full length isoform reconstruction in single cell data, Marie Van Hecke, Ghent University - imec</li>
<li>117. A Novel Computational Pipeline for the Functional Characterization and Deorphanization of G-Protein Coupled Receptors; Catherine Zhou, Stanford University</li>	
<li>118. Parallel and space efficient exact local alignment, Evelin Aasna, Max Planck Institute for Molecular Genetics</li>
<li>119. Pre-Training Dataset Deduplication Improves Genomic LLMs, Mahler Revsine, John Hopkins University</li>	
		</ul>

		</div>
	</div>

</section>
<!-- End Schedule Section -->


