---
layout: post
title: Accepted Papers/Posters of RECOMB-seq 2024
isStaticPost: true
---

<!-- Begin Schedule Section -->
<section id="schedule" class="schedule">
    <div class="content-wrapper text-left">
        <!-- <div class="col-md-8 col-sm-8 col-lg-8 col-lg-offset-2 col-md-offset-2 col-sm-offset-1"> 
        <h2>Program of RECOMB-seq 2024</h2> -->

		<h4>Accepted Papers to the Proceedings Track</h4>
		<ul style="list-style-type: disc;">
		<li>Qixiu Du, May Nee Poon, Xiaocheng Zeng, Pengcheng Zhang, Zheng Wei, Haochen Wang, Ye Wang, Lei Wei and Xiaowo Wang.  Synthetic promoter design in Escherichia coli based on multinomial diffusion model. [ <a href="https://doi.org/10.1016/j.isci.2024.111207">iScience</a> ] </li>
		<li>Mohsen Zakeri, Nathaniel Brown, Omar Ahmed, Travis Gagie and Ben Langmead.  Movi: a fast and cache-efficient full-text pangenome index. [ <a href="https://doi.org/10.1016/j.isci.2024.111464">iScience</a> ] </li>
		<li>Eddie Ferro, Marco Oliva, Travis Gagie and Christina Boucher.  Building a Pangenome Alignment Index via Recursive Prefix-Free Parsing. [ <a href="https://doi.org/10.1016/j.isci.2024.110933">iScience</a> ] </li>
		<li>Francisco Sena, Eliel Ingervo, Shahbaz Khan, Andrey Prjibelski, Sebastian Schmidt and Alexandru Tomescu.  Flowtigs: safety in flow decompositions for assembly graphs. [ <a href="https://doi.org/10.1016/j.isci.2024.111208">iScience</a> ] </li>
		<li>Victor Levallois, Pierre Peterlongo, Yoann Dufresne, Bertrand Le Gal and Francesco Andreace.  The Backpack Quotient Filter: a dynamic and space-efficient data structure for querying k-mers with abundance. [ <a href="https://doi.org/10.1016/j.isci.2024.111435">iScience</a> ] </li>
		<li>Alexander Sasse, Sara Mostafavi and Maria Chikina.  Quick and effective approximation of in silico saturation mutagenesis experiments with first-order Taylor expansion. [ <a href="https://doi.org/10.1016/j.isci.2024.110807">iScience</a> ] </li>
		</ul>
		
		<h4>Accepted Papers to the Overlay Track</h4>
		<ul style="list-style-type: disc;">
		<li>Ilia Minkin and Steven Salzberg.  Quality assessment of splice site annotation based on conservation across multiple species.</li>
		<li>Yunfei Hu, Yikang Li, Manfei Xie, Mingxing Rao, Yichen Henry Liu and Xin Maizie Zhou.  MaskGraphene: Advancing joint embedding, clustering, and batch correction for spatial transcriptomics using graph-based self-supervised learning.</li>
		<li>Tim Dunn, Justin Zook, James Holt and Satish Narayanasamy.  Jointly benchmarking small and structural variant calls with vcfdist.</li>
		<li>Md Hasin Abrar and Paul Medvedev.  PLA-complexity of k-mer multisets.</li>
		<li>Mir Henglin, Maryam Ghareghani, William Harvey, David Porubsky, Sergey Koren, Evan Eichler, Peter Ebert and Tobias Marschall.  Phasing Diploid Genome Assembly Graphs with Single-Cell Strand Sequencing.</li>
		<li>Sergey Vilov and Matthias Heinig.  Investigating the performance of foundation models on human 3’UTR sequences.</li>
		<li>Weiman Yuan, Can Luo, Yunfei Hu, Liting Zhang, Zihang Wen, Yichen Henry Liu, Xian Mallory and Xin Maizie Zhou.  CNVeil enables accurate and robust tumor subclone identification and copy number estimation from single cell sequencing data.
		</ul>

		<h4>Accepted Short Talks</h4>
		<ul style="list-style-type: disc;">
		<li>Mohammad Vahed, Yu Ning Huang, Jiaqi Fu, Kerui Peng and Serghei Mangul. Comprehensive benchmarking of methods to infer from TCR-Seq data.</li>
		<li>Alexis Garretson and Beth L. Dumont. Comprehensive Tissue-Specific Somatic Mutation Profiling via RNA-seq in Diverse Mice.</li>
		<li>Yuhe Cheng and Ludmil B. Alexandrov. DupCaller enables robust detection of somatic mutations from Error-Corrected Sequencing.</li>
		<li>Li Su, Duolin Wang, Fei He and Dong Xu. Integrating scRNA-seq and scTCR-seq data to identify antigen-reactive T cells in T cell repertoire.</li>
		<li>Ying Zhou, Li Song and Heng Li. Full resolution HLA and KIR genes annota)on for human genome assemblies.</li>
		<li>Neng Huang and Heng Li. An efficient and accurate germline SNP caller for long-read RNA sequencing data.</li>
		<li>Hyunho Lee, Kyoungseob Shin, Yongju Lee, Soobin Lee, Seungyoun Lee, Eunjae Lee, Seung Woo Kim, Ha Young Shin, Jong Hoon Kim, Junho Chung and Sunghoon Kwon. Identification of B cell subsets based on antigen receptor sequences using deep learning.</li>
		<li>Arjun Srivatsa and Russell Schwartz. Optimizing Design of Genomics Studies for Clonal Evolution Analysis.</li>
		<li>Kuan-Hao Chao, Jakob M. Heinz, Celine Hoh, Alan Mao, Alaina Shumate, Mihaela Pertea and Steven L Salzberg. Combining DNA and protein alignments to improve genome annotation with LiftOn.</li>
		<li>Lore Depuydt, Luca Renders, Thomas Abeel and Jan Fostier. Pan-genome de Bruijn Graph using the Bidirectional FM-index.</li>
		<li>Ghanshyam Chandra and Chirag Jain. Accelerating whole-genome alignment using parallel chaining algorithm.</li>
		<li>Vikram Shivakumar and Ben Langmead. Mumemto: efficient maximal matching across multiple genomes.</li>
		<li>Stephen Hwang, Nathaniel K. Brown, Omar Y. Ahmed, Katharine Jenike, Sam Kovaka, Michael C. Schatz and Ben Langmead. Compressed Indexing for Pangenome Substring Queries.</li>
		<li>Varuni Sarwal, Seungmo Lee, Jianzhi Yang, Sriram Sankararaman, Mark Chaisson, Eleazar Eskin and Serghei Mangul. VISTA: An integrated framework for structural variant discovery.</li>
		<li>Cecile P G Meier-Scherling, Tavor Baharav, Karamoko Niaré, Julia Salzman, Lorin Crawford and Jeffrey A Bailey. Analyzing the relatedness of genomic variation in malaria parasites using a reference-free approach.</li>
		<li>Ragnar Groot Koerkamp and Pesho Ivanov. A*PA & A*PA2: Up to 20 times faster exact global pairwise alignment.</li>
		</ul>


		<h4>Accepted Posters</h4>
		(NOTE: authors of accepted short-talks are also invited to present a poster.)
		<ul style="list-style-type: disc;">
		<li>Alexis Garretson and Beth L. Dumont.  Genetic Architecture of the Germline Mutation Rate and Reproductive Success in the Collaborative Cross</li>
		<li>Estefania Torrejón, Akiko Teshima, Inês Ferreira, Ana Sofia Carvalho, Hans Christian Beck, Rune Matthiesen, Fabrizia Carli, Amalia Gastaldelli, Maria Paula Macedo and Rita Machado de Oliveira.  Gut derived Extracellular Vesicles Reaching Kupffer Cells: An alternative route for lipid transport out of the gut</li>
		<li>Yunzhe Jiang, Beatrice Borsari and Mark Gerstein.  Comprehensive characterization of pseudogenes across 26 human tissues</li>
		<li>Maria Chikina and Tugrul Balci.  A unified  hypothesis-free feature extraction framework for diverse epigenomic data</li>
		<li>Michelle Wei and Guang Yao.  QDSWorkflow: An Elastic Net-Based Tool for Modeling Cellular Dormancy</li>
		<li>Thi Duyen Do, Ming-Ren Yang and Yu-Wei Wu.  A Unitig-Centered Pan-Genome Approach for Predicting Antibiotic Resistance and Discovering Novel Resistance Genes in Bacterial Strains</li>
		<li>Karishma Chhugani, Serghei Mangul, Oleg Arnaut, Nitesh Sharma and Belin Korukoğlu.  Benchmarking of machine learning algorithms to predict mortality in sepsis from transcriptomic data</li>
		<li>Grigore Boldirev, Nitesh Sharma, Alex Zelikovsky and Serghei Mangul.  Assessing Microbial Genome Representation Across Various Reference Databases: A Comprehensive Evaluation</li>
		<li>Siqin Li, Nicole Genesis Nicole Carpio Paucar and Natalie Farny.  Validating a liquid soil model to explore soil microbial community dynamics</li>
		<li>Dhrithi Deshpande and Serghei Mangul.  Comparative Evaluation of T-Cell Receptor Repertoire Sequencing Methods</li>
		<li>Muhammet Celik, Felix Roman Salcher, Frank Ziemann, Maria Solovey and Maria Colome-Tatche.  Community: A Novel R-Tool for Enhanced Differential Communication Analysis in scRNAseq Data</li>
		<li>Pratik Worah.  Recovering approximate single cell distribution from aggregate measurements</li>
		<li>Dallas Nygard, Julie St-Pierre and Mathieu Lavallée-Adam.  Statistical assessment of gene functional annotation clustering in graph models of chromosome conformation capture data</li>
		<li>Ndivhuwo Nyase, Lebohang Mashatola, Stephanie Muller, Aviwe Kohlakala and Kahn Rhrissorrakrai.  WGTDA: A Topological Framework for Biomarker Discovery in Gene Expression Data</li>
		<li>Timothy Barry, Joseph Deutch, Xihong Lin and Eugene Katsevich.  sceptre: statistically rigorous, computationally efficient, and user-friendly single-cell CRISPR screen data analysis</li>
		<li>Amanda Clare, Wayne Aubrey, Mike Surette and Nicholas Dimonaco.  Partial gene predictions on unassembled reads: evaluating the Good, the Bad and the slightly ORF</li>
		<li>Ragnar Groot Koerkamp and Daniel Liu.  PA-Bench: A framework for benchmarking pairwise aligners</li>
		<li>Daniel Liu and Martin Steinegger.  Block Aligner: an adaptive SIMD-accelerated aligner for sequences and position-specific scoring matrices</li>
		<li>Parishad Mokhber, Alireza Gargoorimotlagh and Babak Khalaj.  scFedVI: A Privacy-Preserving Approach to Mitigating Batch  Effects in Single-Cell RNA-Sequencing Data</li>
		</ul>

	<!-- </div> -->
	</div>
</section>
<!-- End Schedule Section -->


